From 96c35ab2fc8a25ff4c0a78b193ab8d659d55bcbd Mon Sep 17 00:00:00 2001
From: Li Zhou <li.zhou@windriver.com>
Date: Thu, 7 Apr 2022 14:28:38 +0800
Subject: [PATCH] Debian: align config file with stx centos

Porting below commits from stx centos to stx debian for kernel-std:
6fe8d6083263 kernel: Disable NVMe multi-path kconfig option
c9cdb902767d Fixup recent kconfig cleanup
8551799e01ca Resolve v5.10 kernel configuration file differences

Signed-off-by: Li Zhou <li.zhou@windriver.com>
---
 debian/config/amd64/none/config | 660 +++++++++++++-------------------
 1 file changed, 275 insertions(+), 385 deletions(-)

diff --git a/debian/config/amd64/none/config b/debian/config/amd64/none/config
index 771a68b..1f00698 100644
--- a/debian/config/amd64/none/config
+++ b/debian/config/amd64/none/config
@@ -7,7 +7,6 @@ CONFIG_IRQ_WORK=y
 #
 CONFIG_INIT_ENV_ARG_LIMIT=32
 # CONFIG_COMPILE_TEST is not set
-CONFIG_UAPI_HEADER_TEST=y
 CONFIG_LOCALVERSION=""
 # CONFIG_LOCALVERSION_AUTO is not set
 CONFIG_HAVE_KERNEL_GZIP=y
@@ -31,7 +30,7 @@ CONFIG_SYSVIPC=y
 CONFIG_SYSVIPC_SYSCTL=y
 CONFIG_POSIX_MQUEUE=y
 CONFIG_POSIX_MQUEUE_SYSCTL=y
-CONFIG_WATCH_QUEUE=y
+# CONFIG_WATCH_QUEUE is not set
 CONFIG_CROSS_MEMORY_ATTACH=y
 # CONFIG_USELIB is not set
 CONFIG_AUDIT=y
@@ -95,7 +94,7 @@ CONFIG_PREEMPTION=y
 #
 CONFIG_VIRT_CPU_ACCOUNTING=y
 CONFIG_VIRT_CPU_ACCOUNTING_GEN=y
-CONFIG_IRQ_TIME_ACCOUNTING=y
+# CONFIG_IRQ_TIME_ACCOUNTING is not set
 CONFIG_HAVE_SCHED_AVG_IRQ=y
 CONFIG_BSD_PROCESS_ACCT=y
 CONFIG_BSD_PROCESS_ACCT_V3=y
@@ -128,10 +127,10 @@ CONFIG_RCU_NOCB_CPU=y
 
 CONFIG_BUILD_BIN2C=y
 # CONFIG_IKCONFIG is not set
-CONFIG_IKHEADERS=m
+# CONFIG_IKHEADERS is not set
 CONFIG_LOG_BUF_SHIFT=18
 CONFIG_LOG_CPU_MAX_BUF_SHIFT=12
-CONFIG_PRINTK_SAFE_LOG_BUF_SHIFT=12
+CONFIG_PRINTK_SAFE_LOG_BUF_SHIFT=13
 CONFIG_HAVE_UNSTABLE_SCHED_CLOCK=y
 
 #
@@ -189,7 +188,7 @@ CONFIG_RD_XZ=y
 CONFIG_RD_LZO=y
 CONFIG_RD_LZ4=y
 CONFIG_RD_ZSTD=y
-CONFIG_BOOT_CONFIG=y
+# CONFIG_BOOT_CONFIG is not set
 CONFIG_CC_OPTIMIZE_FOR_PERFORMANCE=y
 # CONFIG_CC_OPTIMIZE_FOR_SIZE is not set
 CONFIG_LD_ORPHAN_WARN=y
@@ -312,10 +311,10 @@ CONFIG_X86_X2APIC=y
 CONFIG_X86_MPPARSE=y
 # CONFIG_GOLDFISH is not set
 CONFIG_RETPOLINE=y
-CONFIG_X86_CPU_RESCTRL=y
+# CONFIG_X86_CPU_RESCTRL is not set
 # CONFIG_QEMUX86 is not set
 CONFIG_X86_EXTENDED_PLATFORM=y
-CONFIG_X86_NUMACHIP=y
+# CONFIG_X86_NUMACHIP is not set
 # CONFIG_X86_VSMP is not set
 # CONFIG_X86_UV is not set
 # CONFIG_X86_GOLDFISH is not set
@@ -334,11 +333,11 @@ CONFIG_X86_HV_CALLBACK_VECTOR=y
 # CONFIG_XEN is not set
 CONFIG_KVM_GUEST=y
 CONFIG_ARCH_CPUIDLE_HALTPOLL=y
-CONFIG_PVH=y
+# CONFIG_PVH is not set
 CONFIG_PARAVIRT_TIME_ACCOUNTING=y
 CONFIG_PARAVIRT_CLOCK=y
 # CONFIG_JAILHOUSE_GUEST is not set
-CONFIG_ACRN_GUEST=y
+# CONFIG_ACRN_GUEST is not set
 # CONFIG_MK8 is not set
 # CONFIG_MPSC is not set
 CONFIG_MCORE2=y
@@ -406,7 +405,7 @@ CONFIG_X86_MSR=y
 CONFIG_X86_CPUID=y
 CONFIG_X86_5LEVEL=y
 CONFIG_X86_DIRECT_GBPAGES=y
-CONFIG_X86_CPA_STATISTICS=y
+# CONFIG_X86_CPA_STATISTICS is not set
 CONFIG_AMD_MEM_ENCRYPT=y
 # CONFIG_AMD_MEM_ENCRYPT_ACTIVE_BY_DEFAULT is not set
 CONFIG_NUMA=y
@@ -426,7 +425,7 @@ CONFIG_X86_CHECK_BIOS_CORRUPTION=y
 CONFIG_X86_RESERVE_LOW=64
 CONFIG_MTRR=y
 CONFIG_MTRR_SANITIZER=y
-CONFIG_MTRR_SANITIZER_ENABLE_DEFAULT=0
+CONFIG_MTRR_SANITIZER_ENABLE_DEFAULT=1
 CONFIG_MTRR_SANITIZER_SPARE_REG_NR_DEFAULT=1
 CONFIG_X86_PAT=y
 CONFIG_ARCH_USES_PG_UNCACHED=y
@@ -449,16 +448,14 @@ CONFIG_SCHED_HRTICK=y
 CONFIG_KEXEC=y
 CONFIG_KEXEC_FILE=y
 CONFIG_ARCH_HAS_KEXEC_PURGATORY=y
-CONFIG_KEXEC_SIG=y
-# CONFIG_KEXEC_SIG_FORCE is not set
-CONFIG_KEXEC_BZIMAGE_VERIFY_SIG=y
+# CONFIG_KEXEC_SIG is not set
 CONFIG_CRASH_DUMP=y
 CONFIG_KEXEC_JUMP=y
 CONFIG_PHYSICAL_START=0x1000000
 CONFIG_RELOCATABLE=y
 CONFIG_RANDOMIZE_BASE=y
 CONFIG_X86_NEED_RELOCS=y
-CONFIG_PHYSICAL_ALIGN=0x1000000
+CONFIG_PHYSICAL_ALIGN=0x200000
 CONFIG_DYNAMIC_MEMORY_LAYOUT=y
 CONFIG_RANDOMIZE_MEMORY=y
 CONFIG_RANDOMIZE_MEMORY_PHYSICAL_PADDING=0x0
@@ -503,7 +500,7 @@ CONFIG_PM_SLEEP_DEBUG=y
 # CONFIG_PM_TRACE_RTC is not set
 CONFIG_PM_CLK=y
 # CONFIG_WQ_POWER_EFFICIENT_DEFAULT is not set
-CONFIG_ENERGY_MODEL=y
+# CONFIG_ENERGY_MODEL is not set
 CONFIG_ARCH_SUPPORTS_ACPI=y
 CONFIG_ACPI=y
 CONFIG_ACPI_LEGACY_TABLES_LOOKUP=y
@@ -544,7 +541,7 @@ CONFIG_ACPI_BGRT=y
 CONFIG_ACPI_NFIT=m
 # CONFIG_NFIT_SECURITY_DEBUG is not set
 CONFIG_ACPI_NUMA=y
-CONFIG_ACPI_HMAT=y
+# CONFIG_ACPI_HMAT is not set
 CONFIG_HAVE_ACPI_APEI=y
 CONFIG_HAVE_ACPI_APEI_NMI=y
 CONFIG_ACPI_APEI=y
@@ -553,9 +550,7 @@ CONFIG_ACPI_APEI_PCIEAER=y
 CONFIG_ACPI_APEI_MEMORY_FAILURE=y
 CONFIG_ACPI_APEI_EINJ=m
 # CONFIG_ACPI_APEI_ERST_DEBUG is not set
-CONFIG_ACPI_DPTF=y
-CONFIG_DPTF_POWER=m
-CONFIG_DPTF_PCH_FIVR=m
+# CONFIG_ACPI_DPTF is not set
 CONFIG_ACPI_WATCHDOG=y
 # CONFIG_ACPI_CONFIGFS is not set
 CONFIG_PMIC_OPREGION=y
@@ -656,23 +651,21 @@ CONFIG_EFI_VARS_PSTORE=y
 CONFIG_EFI_VARS_PSTORE_DEFAULT_DISABLE=y
 CONFIG_EFI_RUNTIME_MAP=y
 # CONFIG_EFI_FAKE_MEMMAP is not set
-CONFIG_EFI_SOFT_RESERVE=y
 CONFIG_EFI_RUNTIME_WRAPPERS=y
 CONFIG_EFI_GENERIC_STUB_INITRD_CMDLINE_LOADER=y
 # CONFIG_EFI_BOOTLOADER_CONTROL is not set
 # CONFIG_EFI_CAPSULE_LOADER is not set
 # CONFIG_EFI_TEST is not set
-CONFIG_APPLE_PROPERTIES=y
+# CONFIG_APPLE_PROPERTIES is not set
 # CONFIG_RESET_ATTACK_MITIGATION is not set
-CONFIG_EFI_RCI2_TABLE=y
+# CONFIG_EFI_RCI2_TABLE is not set
 # CONFIG_EFI_DISABLE_PCI_DMA is not set
 # end of EFI (Extensible Firmware Interface) Support
 
 CONFIG_UEFI_CPER=y
 CONFIG_UEFI_CPER_X86=y
-CONFIG_EFI_DEV_PATH_PARSER=y
 CONFIG_EFI_EARLYCON=y
-# CONFIG_EFI_CUSTOM_SSDT_OVERLAYS is not set
+CONFIG_EFI_CUSTOM_SSDT_OVERLAYS=y
 
 #
 # Tegra firmware driver
@@ -840,33 +833,30 @@ CONFIG_MODULE_SIG_HASH="sha256"
 # CONFIG_TRIM_UNUSED_KSYMS is not set
 CONFIG_MODULES_TREE_LOOKUP=y
 CONFIG_BLOCK=y
-CONFIG_BLK_RQ_ALLOC_TIME=y
 CONFIG_BLK_SCSI_REQUEST=y
 CONFIG_BLK_CGROUP_RWSTAT=y
 CONFIG_BLK_DEV_BSG=y
 CONFIG_BLK_DEV_BSGLIB=y
 CONFIG_BLK_DEV_INTEGRITY=y
 CONFIG_BLK_DEV_INTEGRITY_T10=m
-CONFIG_BLK_DEV_ZONED=y
+# CONFIG_BLK_DEV_ZONED is not set
 CONFIG_BLK_DEV_THROTTLING=y
 # CONFIG_BLK_DEV_THROTTLING_LOW is not set
 # CONFIG_BLK_CMDLINE_PARSER is not set
 CONFIG_BLK_WBT=y
 # CONFIG_BLK_CGROUP_IOLATENCY is not set
-CONFIG_BLK_CGROUP_IOCOST=y
+# CONFIG_BLK_CGROUP_IOCOST is not set
 CONFIG_BLK_WBT_MQ=y
 CONFIG_BLK_DEBUG_FS=y
-CONFIG_BLK_DEBUG_FS_ZONED=y
 # CONFIG_BLK_SED_OPAL is not set
-CONFIG_BLK_INLINE_ENCRYPTION=y
-# CONFIG_BLK_INLINE_ENCRYPTION_FALLBACK is not set
+# CONFIG_BLK_INLINE_ENCRYPTION is not set
 
 #
 # Partition Types
 #
 CONFIG_PARTITION_ADVANCED=y
 # CONFIG_ACORN_PARTITION is not set
-CONFIG_AIX_PARTITION=y
+# CONFIG_AIX_PARTITION is not set
 # CONFIG_OSF_PARTITION is not set
 # CONFIG_AMIGA_PARTITION is not set
 # CONFIG_ATARI_PARTITION is not set
@@ -876,8 +866,7 @@ CONFIG_MSDOS_PARTITION=y
 # CONFIG_MINIX_SUBPARTITION is not set
 # CONFIG_SOLARIS_X86_PARTITION is not set
 # CONFIG_UNIXWARE_DISKLABEL is not set
-CONFIG_LDM_PARTITION=y
-# CONFIG_LDM_DEBUG is not set
+# CONFIG_LDM_PARTITION is not set
 # CONFIG_SGI_PARTITION is not set
 # CONFIG_ULTRIX_PARTITION is not set
 # CONFIG_SUN_PARTITION is not set
@@ -958,7 +947,7 @@ CONFIG_BOUNCE=y
 CONFIG_VIRT_TO_BUS=y
 CONFIG_MMU_NOTIFIER=y
 CONFIG_KSM=y
-CONFIG_DEFAULT_MMAP_MIN_ADDR=65536
+CONFIG_DEFAULT_MMAP_MIN_ADDR=4096
 CONFIG_ARCH_SUPPORTS_MEMORY_FAILURE=y
 CONFIG_MEMORY_FAILURE=y
 CONFIG_HWPOISON_INJECT=m
@@ -983,7 +972,7 @@ CONFIG_ZSWAP_ZPOOL_DEFAULT="zbud"
 # CONFIG_ZSWAP_DEFAULT_ON is not set
 CONFIG_ZPOOL=y
 CONFIG_ZBUD=y
-CONFIG_Z3FOLD=y
+# CONFIG_Z3FOLD is not set
 CONFIG_ZSMALLOC=y
 # CONFIG_ZSMALLOC_STAT is not set
 CONFIG_GENERIC_EARLY_IOREMAP=y
@@ -1029,10 +1018,9 @@ CONFIG_XFRM_ESP=m
 CONFIG_XFRM_IPCOMP=m
 CONFIG_NET_KEY=m
 CONFIG_NET_KEY_MIGRATE=y
-CONFIG_XFRM_ESPINTCP=y
 # CONFIG_SMC is not set
 CONFIG_XDP_SOCKETS=y
-CONFIG_XDP_SOCKETS_DIAG=m
+# CONFIG_XDP_SOCKETS_DIAG is not set
 CONFIG_INET=y
 CONFIG_IP_MULTICAST=y
 CONFIG_IP_ADVANCED_ROUTER=y
@@ -1060,7 +1048,7 @@ CONFIG_NET_UDP_TUNNEL=m
 CONFIG_INET_AH=m
 CONFIG_INET_ESP=m
 CONFIG_INET_ESP_OFFLOAD=m
-CONFIG_INET_ESPINTCP=y
+# CONFIG_INET_ESPINTCP is not set
 CONFIG_INET_IPCOMP=m
 CONFIG_INET_XFRM_TUNNEL=m
 CONFIG_INET_TUNNEL=m
@@ -1097,7 +1085,7 @@ CONFIG_IPV6_OPTIMISTIC_DAD=y
 CONFIG_INET6_AH=m
 CONFIG_INET6_ESP=m
 CONFIG_INET6_ESP_OFFLOAD=m
-CONFIG_INET6_ESPINTCP=y
+# CONFIG_INET6_ESPINTCP is not set
 CONFIG_INET6_IPCOMP=m
 CONFIG_IPV6_MIP6=m
 # CONFIG_IPV6_ILA is not set
@@ -1119,9 +1107,7 @@ CONFIG_IPV6_SEG6_LWTUNNEL=y
 CONFIG_IPV6_SEG6_BPF=y
 CONFIG_IPV6_RPL_LWTUNNEL=y
 CONFIG_NETLABEL=y
-CONFIG_MPTCP=y
-CONFIG_INET_MPTCP_DIAG=m
-CONFIG_MPTCP_IPV6=y
+# CONFIG_MPTCP is not set
 CONFIG_NETWORK_SECMARK=y
 CONFIG_NET_PTP_CLASSIFY=y
 CONFIG_NETWORK_PHY_TIMESTAMPING=y
@@ -1192,7 +1178,7 @@ CONFIG_NFT_LIMIT=m
 CONFIG_NFT_MASQ=m
 CONFIG_NFT_REDIR=m
 CONFIG_NFT_NAT=m
-CONFIG_NFT_TUNNEL=m
+# CONFIG_NFT_TUNNEL is not set
 CONFIG_NFT_OBJREF=m
 CONFIG_NFT_QUEUE=m
 CONFIG_NFT_QUOTA=m
@@ -1202,11 +1188,11 @@ CONFIG_NFT_COMPAT=m
 CONFIG_NFT_HASH=m
 CONFIG_NFT_FIB=m
 CONFIG_NFT_FIB_INET=m
-CONFIG_NFT_XFRM=m
+# CONFIG_NFT_XFRM is not set
 # CONFIG_NFT_SOCKET is not set
 # CONFIG_NFT_OSF is not set
-CONFIG_NFT_TPROXY=m
-CONFIG_NFT_SYNPROXY=m
+# CONFIG_NFT_TPROXY is not set
+# CONFIG_NFT_SYNPROXY is not set
 CONFIG_NF_DUP_NETDEV=m
 CONFIG_NFT_DUP_NETDEV=m
 CONFIG_NFT_FWD_NETDEV=m
@@ -1451,7 +1437,7 @@ CONFIG_NF_TABLES_BRIDGE=m
 CONFIG_NFT_BRIDGE_META=m
 CONFIG_NFT_BRIDGE_REJECT=m
 CONFIG_NF_LOG_BRIDGE=m
-CONFIG_NF_CONNTRACK_BRIDGE=m
+# CONFIG_NF_CONNTRACK_BRIDGE is not set
 CONFIG_BRIDGE_NF_EBTABLES=m
 CONFIG_BRIDGE_EBT_BROUTE=m
 CONFIG_BRIDGE_EBT_T_FILTER=m
@@ -1488,7 +1474,7 @@ CONFIG_TIPC=m
 CONFIG_TIPC_MEDIA_UDP=y
 CONFIG_TIPC_CRYPTO=y
 CONFIG_TIPC_DIAG=m
-CONFIG_ATM=y
+CONFIG_ATM=m
 CONFIG_ATM_CLIP=m
 # CONFIG_ATM_CLIP_NO_ICMP is not set
 CONFIG_ATM_LANE=m
@@ -1506,7 +1492,7 @@ CONFIG_MRP=m
 CONFIG_BRIDGE=m
 CONFIG_BRIDGE_IGMP_SNOOPING=y
 CONFIG_BRIDGE_VLAN_FILTERING=y
-CONFIG_BRIDGE_MRP=y
+# CONFIG_BRIDGE_MRP is not set
 CONFIG_HAVE_NET_DSA=y
 # CONFIG_NET_DSA is not set
 CONFIG_VLAN_8021Q=m
@@ -1563,7 +1549,7 @@ CONFIG_NET_SCH_PIE=m
 # CONFIG_NET_SCH_FQ_PIE is not set
 CONFIG_NET_SCH_INGRESS=m
 CONFIG_NET_SCH_PLUG=m
-CONFIG_NET_SCH_ETS=m
+# CONFIG_NET_SCH_ETS is not set
 CONFIG_NET_SCH_DEFAULT=y
 # CONFIG_DEFAULT_CODEL is not set
 CONFIG_DEFAULT_FQ_CODEL=y
@@ -1611,16 +1597,16 @@ CONFIG_NET_ACT_PEDIT=m
 CONFIG_NET_ACT_SIMP=m
 CONFIG_NET_ACT_SKBEDIT=m
 CONFIG_NET_ACT_CSUM=m
-CONFIG_NET_ACT_MPLS=m
+# CONFIG_NET_ACT_MPLS is not set
 CONFIG_NET_ACT_VLAN=m
 CONFIG_NET_ACT_BPF=m
 # CONFIG_NET_ACT_CONNMARK is not set
-CONFIG_NET_ACT_CTINFO=m
+# CONFIG_NET_ACT_CTINFO is not set
 CONFIG_NET_ACT_SKBMOD=m
 # CONFIG_NET_ACT_IFE is not set
 CONFIG_NET_ACT_TUNNEL_KEY=m
-CONFIG_NET_ACT_GATE=m
-CONFIG_NET_TC_SKB_EXT=y
+# CONFIG_NET_ACT_GATE is not set
+# CONFIG_NET_TC_SKB_EXT is not set
 CONFIG_NET_SCH_FIFO=y
 CONFIG_DCB=y
 CONFIG_DNS_RESOLVER=m
@@ -1643,9 +1629,7 @@ CONFIG_NET_NSH=y
 # CONFIG_HSR is not set
 CONFIG_NET_SWITCHDEV=y
 CONFIG_NET_L3_MASTER_DEV=y
-CONFIG_QRTR=m
-# CONFIG_QRTR_TUN is not set
-# CONFIG_QRTR_MHI is not set
+# CONFIG_QRTR is not set
 # CONFIG_NET_NCSI is not set
 CONFIG_RPS=y
 CONFIG_RFS_ACCEL=y
@@ -1719,7 +1703,7 @@ CONFIG_CAN_EMS_USB=m
 CONFIG_CAN_ESD_USB2=m
 # CONFIG_CAN_GS_USB is not set
 CONFIG_CAN_KVASER_USB=m
-CONFIG_CAN_MCBA_USB=m
+# CONFIG_CAN_MCBA_USB is not set
 CONFIG_CAN_PEAK_USB=m
 # CONFIG_CAN_UCAN is not set
 # end of CAN USB interfaces
@@ -1785,7 +1769,7 @@ CONFIG_PCIEASPM_DEFAULT=y
 CONFIG_PCIE_PME=y
 CONFIG_PCIE_DPC=y
 # CONFIG_PCIE_PTM is not set
-CONFIG_PCIE_EDR=y
+# CONFIG_PCIE_EDR is not set
 CONFIG_PCI_MSI=y
 CONFIG_PCI_MSI_IRQ_DOMAIN=y
 CONFIG_PCI_QUIRKS=y
@@ -1809,7 +1793,7 @@ CONFIG_HOTPLUG_PCI_ACPI=y
 #
 # PCI controller drivers
 #
-CONFIG_VMD=m
+CONFIG_VMD=y
 CONFIG_PCI_HYPERV_INTERFACE=m
 
 #
@@ -1848,6 +1832,7 @@ CONFIG_PCI_HYPERV_INTERFACE=m
 #
 # Generic Driver Options
 #
+CONFIG_AUXILIARY_BUS=y
 # CONFIG_UEVENT_HELPER is not set
 CONFIG_DEVTMPFS=y
 CONFIG_DEVTMPFS_MOUNT=y
@@ -1858,10 +1843,9 @@ CONFIG_PREVENT_FIRMWARE_BUILD=y
 # Firmware loader
 #
 CONFIG_FW_LOADER=y
-CONFIG_FW_LOADER_PAGED_BUF=y
 CONFIG_EXTRA_FIRMWARE=""
 # CONFIG_FW_LOADER_USER_HELPER is not set
-CONFIG_FW_LOADER_COMPRESS=y
+# CONFIG_FW_LOADER_COMPRESS is not set
 CONFIG_FW_CACHE=y
 # end of Firmware loader
 
@@ -1869,7 +1853,6 @@ CONFIG_ALLOW_DEV_COREDUMP=y
 # CONFIG_DEBUG_DRIVER is not set
 # CONFIG_DEBUG_DEVRES is not set
 # CONFIG_DEBUG_TEST_DRIVER_REMOVE is not set
-CONFIG_HMEM_REPORTING=y
 # CONFIG_TEST_ASYNC_DRIVER_PROBE is not set
 CONFIG_GENERIC_CPU_AUTOPROBE=y
 CONFIG_GENERIC_CPU_VULNERABILITIES=y
@@ -1877,7 +1860,6 @@ CONFIG_REGMAP=y
 CONFIG_REGMAP_I2C=y
 CONFIG_REGMAP_SPI=m
 CONFIG_REGMAP_MMIO=y
-CONFIG_REGMAP_SPI_AVMM=m
 CONFIG_DMA_SHARED_BUFFER=y
 # CONFIG_DMA_FENCE_TRACE is not set
 # end of Generic Driver Options
@@ -1885,17 +1867,12 @@ CONFIG_DMA_SHARED_BUFFER=y
 #
 # Bus devices
 #
-CONFIG_MHI_BUS=m
-# CONFIG_MHI_BUS_DEBUG is not set
+# CONFIG_MHI_BUS is not set
 # end of Bus devices
 
 CONFIG_CONNECTOR=y
 CONFIG_PROC_EVENTS=y
-CONFIG_GNSS=m
-CONFIG_GNSS_SERIAL=m
-CONFIG_GNSS_MTK_SERIAL=m
-CONFIG_GNSS_SIRF_SERIAL=m
-CONFIG_GNSS_UBX_SERIAL=m
+# CONFIG_GNSS is not set
 CONFIG_MTD=m
 # CONFIG_MTD_TESTS is not set
 
@@ -2032,13 +2009,13 @@ CONFIG_BLK_DEV_RBD=m
 #
 CONFIG_NVME_CORE=m
 CONFIG_BLK_DEV_NVME=m
-CONFIG_NVME_MULTIPATH=y
-CONFIG_NVME_HWMON=y
+# CONFIG_NVME_MULTIPATH is not set
+# CONFIG_NVME_HWMON is not set
 CONFIG_NVME_FABRICS=m
 CONFIG_NVME_FC=m
 CONFIG_NVME_TCP=m
 CONFIG_NVME_TARGET=m
-CONFIG_NVME_TARGET_PASSTHRU=y
+# CONFIG_NVME_TARGET_PASSTHRU is not set
 CONFIG_NVME_TARGET_LOOP=m
 CONFIG_NVME_TARGET_FC=m
 CONFIG_NVME_TARGET_FCLOOP=m
@@ -2079,10 +2056,10 @@ CONFIG_EEPROM_AT24=m
 # CONFIG_EEPROM_AT25 is not set
 CONFIG_EEPROM_LEGACY=m
 # CONFIG_EEPROM_MAX6875 is not set
-CONFIG_EEPROM_93CX6=m
+# CONFIG_EEPROM_93CX6 is not set
 # CONFIG_EEPROM_93XX46 is not set
 # CONFIG_EEPROM_IDT_89HPESX is not set
-CONFIG_EEPROM_EE1004=m
+# CONFIG_EEPROM_EE1004 is not set
 # end of EEPROM support
 
 # CONFIG_CB710_CORE is not set
@@ -2098,15 +2075,15 @@ CONFIG_EEPROM_EE1004=m
 CONFIG_INTEL_MEI=m
 CONFIG_INTEL_MEI_ME=m
 # CONFIG_INTEL_MEI_TXE is not set
-CONFIG_INTEL_MEI_HDCP=m
+# CONFIG_INTEL_MEI_HDCP is not set
 # CONFIG_VMWARE_VMCI is not set
 # CONFIG_GENWQE is not set
 # CONFIG_ECHO is not set
-CONFIG_MISC_ALCOR_PCI=m
+# CONFIG_MISC_ALCOR_PCI is not set
 CONFIG_MISC_RTSX_PCI=m
 CONFIG_MISC_RTSX_USB=m
 # CONFIG_HABANA_AI is not set
-CONFIG_UACCE=m
+# CONFIG_UACCE is not set
 # end of Misc devices
 
 CONFIG_HAVE_IDE=y
@@ -2184,8 +2161,8 @@ CONFIG_SCSI_SMARTPQI=y
 # CONFIG_SCSI_UFSHCD is not set
 CONFIG_SCSI_HPTIOP=y
 # CONFIG_SCSI_BUSLOGIC is not set
-CONFIG_SCSI_MYRB=m
-CONFIG_SCSI_MYRS=m
+# CONFIG_SCSI_MYRB is not set
+# CONFIG_SCSI_MYRS is not set
 # CONFIG_VMWARE_PVSCSI is not set
 CONFIG_HYPERV_STORAGE=m
 CONFIG_LIBFC=m
@@ -2194,8 +2171,7 @@ CONFIG_LIBFCOE=m
 # CONFIG_FCOE_FNIC is not set
 # CONFIG_SCSI_SNIC is not set
 # CONFIG_SCSI_DMX3191D is not set
-CONFIG_SCSI_FDOMAIN=m
-CONFIG_SCSI_FDOMAIN_PCI=m
+# CONFIG_SCSI_FDOMAIN_PCI is not set
 CONFIG_SCSI_GDTH=y
 CONFIG_SCSI_ISCI=y
 CONFIG_SCSI_IPS=y
@@ -2241,7 +2217,7 @@ CONFIG_SATA_PMP=y
 # Controllers with non-SFF native interface
 #
 CONFIG_SATA_AHCI=m
-CONFIG_SATA_MOBILE_LPM_POLICY=3
+CONFIG_SATA_MOBILE_LPM_POLICY=0
 CONFIG_SATA_AHCI_PLATFORM=m
 # CONFIG_SATA_INIC162X is not set
 # CONFIG_SATA_ACARD_AHCI is not set
@@ -2331,7 +2307,7 @@ CONFIG_MD_RAID0=m
 CONFIG_MD_RAID1=m
 CONFIG_MD_RAID10=m
 CONFIG_MD_RAID456=m
-CONFIG_MD_MULTIPATH=m
+# CONFIG_MD_MULTIPATH is not set
 CONFIG_MD_FAULTY=m
 # CONFIG_MD_CLUSTER is not set
 # CONFIG_BCACHE is not set
@@ -2349,9 +2325,9 @@ CONFIG_DM_THIN_PROVISIONING=m
 CONFIG_DM_CACHE=m
 CONFIG_DM_CACHE_SMQ=m
 CONFIG_DM_WRITECACHE=m
-CONFIG_DM_EBS=m
+# CONFIG_DM_EBS is not set
 # CONFIG_DM_ERA is not set
-CONFIG_DM_CLONE=m
+# CONFIG_DM_CLONE is not set
 CONFIG_DM_MIRROR=m
 CONFIG_DM_LOG_USERSPACE=m
 # CONFIG_DM_RAID is not set
@@ -2359,18 +2335,17 @@ CONFIG_DM_ZERO=m
 CONFIG_DM_MULTIPATH=m
 CONFIG_DM_MULTIPATH_QL=m
 CONFIG_DM_MULTIPATH_ST=m
-CONFIG_DM_MULTIPATH_HST=m
+# CONFIG_DM_MULTIPATH_HST is not set
 CONFIG_DM_DELAY=m
-CONFIG_DM_DUST=m
+# CONFIG_DM_DUST is not set
 CONFIG_DM_UEVENT=y
 CONFIG_DM_FLAKEY=m
 CONFIG_DM_VERITY=m
-CONFIG_DM_VERITY_VERIFY_ROOTHASH_SIG=y
+# CONFIG_DM_VERITY_VERIFY_ROOTHASH_SIG is not set
 # CONFIG_DM_VERITY_FEC is not set
 # CONFIG_DM_SWITCH is not set
 CONFIG_DM_LOG_WRITES=m
 CONFIG_DM_INTEGRITY=m
-CONFIG_DM_ZONED=m
 CONFIG_TARGET_CORE=m
 CONFIG_TCM_IBLOCK=m
 CONFIG_TCM_FILEIO=m
@@ -2400,8 +2375,7 @@ CONFIG_MII=m
 CONFIG_NET_CORE=y
 CONFIG_BONDING=m
 CONFIG_DUMMY=m
-CONFIG_WIREGUARD=m
-# CONFIG_WIREGUARD_DEBUG is not set
+# CONFIG_WIREGUARD is not set
 # CONFIG_EQUALIZER is not set
 # CONFIG_NET_FC is not set
 CONFIG_IFB=m
@@ -2418,7 +2392,7 @@ CONFIG_IPVLAN=m
 CONFIG_IPVTAP=m
 CONFIG_VXLAN=m
 # CONFIG_GENEVE is not set
-CONFIG_BAREUDP=m
+# CONFIG_BAREUDP is not set
 # CONFIG_GTP is not set
 CONFIG_MACSEC=m
 CONFIG_NETCONSOLE=m
@@ -2469,7 +2443,7 @@ CONFIG_NET_VENDOR_ATHEROS=y
 # CONFIG_NET_VENDOR_AURORA is not set
 CONFIG_NET_VENDOR_BROADCOM=y
 # CONFIG_B44 is not set
-CONFIG_BCMGENET=m
+# CONFIG_BCMGENET is not set
 CONFIG_BNX2=m
 CONFIG_CNIC=m
 CONFIG_TIGON3=m
@@ -2517,7 +2491,8 @@ CONFIG_BE2NET_HWMON=y
 CONFIG_BE2NET_LANCER=y
 CONFIG_BE2NET_SKYHAWK=y
 # CONFIG_NET_VENDOR_EZCHIP is not set
-# CONFIG_NET_VENDOR_GOOGLE is not set
+CONFIG_NET_VENDOR_GOOGLE=y
+CONFIG_GVE=m
 CONFIG_NET_VENDOR_HUAWEI=y
 CONFIG_HINIC=m
 # CONFIG_NET_VENDOR_I825XX is not set
@@ -2550,7 +2525,7 @@ CONFIG_NET_VENDOR_MICREL=y
 # CONFIG_KS8842 is not set
 # CONFIG_KS8851 is not set
 # CONFIG_KS8851_MLL is not set
-CONFIG_KSZ884X_PCI=m
+# CONFIG_KSZ884X_PCI is not set
 # CONFIG_NET_VENDOR_MICROCHIP is not set
 # CONFIG_NET_VENDOR_MICROSEMI is not set
 CONFIG_NET_VENDOR_MYRI=y
@@ -2558,9 +2533,8 @@ CONFIG_NET_VENDOR_MYRI=y
 # CONFIG_FEALNX is not set
 # CONFIG_NET_VENDOR_NATSEMI is not set
 CONFIG_NET_VENDOR_NETERION=y
-CONFIG_S2IO=m
-CONFIG_VXGE=m
-# CONFIG_VXGE_DEBUG_TRACE_ALL is not set
+# CONFIG_S2IO is not set
+# CONFIG_VXGE is not set
 CONFIG_NET_VENDOR_NETRONOME=y
 CONFIG_NFP=m
 CONFIG_NFP_APP_FLOWER=y
@@ -2574,7 +2548,7 @@ CONFIG_NET_VENDOR_PACKET_ENGINES=y
 # CONFIG_HAMACHI is not set
 # CONFIG_YELLOWFIN is not set
 CONFIG_NET_VENDOR_PENSANDO=y
-CONFIG_IONIC=m
+# CONFIG_IONIC is not set
 CONFIG_NET_VENDOR_QLOGIC=y
 CONFIG_QLA3XXX=m
 # CONFIG_QLCNIC is not set
@@ -2618,7 +2592,7 @@ CONFIG_NET_VENDOR_SOLARFLARE=y
 CONFIG_NET_VENDOR_XILINX=y
 # CONFIG_XILINX_EMACLITE is not set
 # CONFIG_XILINX_AXI_EMAC is not set
-CONFIG_XILINX_LL_TEMAC=m
+# CONFIG_XILINX_LL_TEMAC is not set
 # CONFIG_FDDI is not set
 # CONFIG_HIPPI is not set
 # CONFIG_NET_SB1000 is not set
@@ -2631,11 +2605,11 @@ CONFIG_FIXED_PHY=y
 # MII PHY device drivers
 #
 CONFIG_AMD_PHY=m
-CONFIG_ADIN_PHY=m
+# CONFIG_ADIN_PHY is not set
 CONFIG_AQUANTIA_PHY=m
-CONFIG_AX88796B_PHY=m
+# CONFIG_AX88796B_PHY is not set
 CONFIG_BROADCOM_PHY=m
-CONFIG_BCM54140_PHY=m
+# CONFIG_BCM54140_PHY is not set
 CONFIG_BCM7XXX_PHY=m
 # CONFIG_BCM84881_PHY is not set
 # CONFIG_BCM87XX_PHY is not set
@@ -2666,7 +2640,7 @@ CONFIG_DP83822_PHY=m
 # CONFIG_DP83TC811_PHY is not set
 CONFIG_DP83848_PHY=m
 # CONFIG_DP83867_PHY is not set
-CONFIG_DP83869_PHY=m
+# CONFIG_DP83869_PHY is not set
 # CONFIG_VITESSE_PHY is not set
 CONFIG_XILINX_GMII2RGMII=m
 CONFIG_MICREL_KS8995MA=m
@@ -2675,7 +2649,7 @@ CONFIG_MDIO_BUS=y
 CONFIG_MDIO_DEVRES=y
 # CONFIG_MDIO_BITBANG is not set
 CONFIG_MDIO_BCM_UNIMAC=m
-CONFIG_MDIO_MVUSB=m
+# CONFIG_MDIO_MVUSB is not set
 # CONFIG_MDIO_MSCC_MIIM is not set
 # CONFIG_MDIO_THUNDER is not set
 
@@ -2686,7 +2660,7 @@ CONFIG_MDIO_MVUSB=m
 #
 # PCS device drivers
 #
-CONFIG_PCS_XPCS=m
+# CONFIG_PCS_XPCS is not set
 # end of PCS device drivers
 
 CONFIG_PPP=m
@@ -2753,7 +2727,6 @@ CONFIG_IEEE802154_FAKELB=m
 # CONFIG_IEEE802154_HWSIM is not set
 CONFIG_VMXNET3=m
 CONFIG_FUJITSU_ES=m
-CONFIG_USB4_NET=m
 CONFIG_HYPERV_NET=m
 CONFIG_NETDEVSIM=m
 CONFIG_NET_FAILOVER=m
@@ -2765,7 +2738,7 @@ CONFIG_NET_FAILOVER=m
 #
 CONFIG_INPUT=y
 CONFIG_INPUT_LEDS=y
-CONFIG_INPUT_FF_MEMLESS=y
+CONFIG_INPUT_FF_MEMLESS=m
 # CONFIG_INPUT_POLLDEV is not set
 # CONFIG_INPUT_SPARSEKMAP is not set
 # CONFIG_INPUT_MATRIXKMAP is not set
@@ -2788,9 +2761,9 @@ CONFIG_INPUT_KEYBOARD=y
 # CONFIG_KEYBOARD_ADC is not set
 # CONFIG_KEYBOARD_ADP5588 is not set
 # CONFIG_KEYBOARD_ADP5589 is not set
-CONFIG_KEYBOARD_APPLESPI=m
+# CONFIG_KEYBOARD_APPLESPI is not set
 CONFIG_KEYBOARD_ATKBD=y
-CONFIG_KEYBOARD_QT1050=m
+# CONFIG_KEYBOARD_QT1050 is not set
 # CONFIG_KEYBOARD_QT1070 is not set
 # CONFIG_KEYBOARD_QT2160 is not set
 # CONFIG_KEYBOARD_DLINK_DIR685 is not set
@@ -2851,7 +2824,7 @@ CONFIG_INPUT_MISC=y
 # CONFIG_INPUT_APANEL is not set
 # CONFIG_INPUT_GPIO_BEEPER is not set
 # CONFIG_INPUT_GPIO_DECODER is not set
-CONFIG_INPUT_GPIO_VIBRA=m
+# CONFIG_INPUT_GPIO_VIBRA is not set
 # CONFIG_INPUT_ATLAS_BTNS is not set
 # CONFIG_INPUT_ATI_REMOTE2 is not set
 # CONFIG_INPUT_KEYSPAN_REMOTE is not set
@@ -2866,7 +2839,7 @@ CONFIG_INPUT_GPIO_VIBRA=m
 CONFIG_INPUT_GPIO_ROTARY_ENCODER=m
 # CONFIG_INPUT_ADXL34X is not set
 # CONFIG_INPUT_IMS_PCU is not set
-CONFIG_INPUT_IQS269A=m
+# CONFIG_INPUT_IQS269A is not set
 # CONFIG_INPUT_CMA3000 is not set
 # CONFIG_INPUT_IDEAPAD_SLIDEBAR is not set
 # CONFIG_INPUT_DRV260X_HAPTICS is not set
@@ -2883,7 +2856,7 @@ CONFIG_RMI4_F11=y
 CONFIG_RMI4_F12=y
 CONFIG_RMI4_F30=y
 CONFIG_RMI4_F34=y
-CONFIG_RMI4_F3A=y
+# CONFIG_RMI4_F3A is not set
 CONFIG_RMI4_F55=y
 
 #
@@ -2935,7 +2908,7 @@ CONFIG_SERIAL_8250_DMA=y
 CONFIG_SERIAL_8250_PCI=y
 CONFIG_SERIAL_8250_EXAR=y
 CONFIG_SERIAL_8250_NR_UARTS=32
-CONFIG_SERIAL_8250_RUNTIME_UARTS=32
+CONFIG_SERIAL_8250_RUNTIME_UARTS=4
 CONFIG_SERIAL_8250_EXTENDED=y
 CONFIG_SERIAL_8250_MANY_PORTS=y
 CONFIG_SERIAL_8250_SHARE_IRQ=y
@@ -2971,7 +2944,7 @@ CONFIG_SERIAL_CORE_CONSOLE=y
 
 CONFIG_SERIAL_MCTRL_GPIO=y
 CONFIG_SERIAL_NONSTANDARD=y
-CONFIG_ROCKETPORT=m
+# CONFIG_ROCKETPORT is not set
 CONFIG_CYCLADES=m
 # CONFIG_CYZ_INTR is not set
 # CONFIG_MOXA_INTELLIO is not set
@@ -2983,11 +2956,10 @@ CONFIG_SYNCLINK_GT=m
 CONFIG_N_HDLC=m
 # CONFIG_N_GSM is not set
 # CONFIG_NOZOMI is not set
-CONFIG_NULL_TTY=m
+# CONFIG_NULL_TTY is not set
 # CONFIG_TRACE_SINK is not set
 CONFIG_HVC_DRIVER=y
-CONFIG_SERIAL_DEV_BUS=y
-CONFIG_SERIAL_DEV_CTRL_TTYPORT=y
+# CONFIG_SERIAL_DEV_BUS is not set
 CONFIG_VIRTIO_CONSOLE=m
 CONFIG_IPMI_HANDLER=m
 CONFIG_IPMI_DMI_DECODE=y
@@ -3006,7 +2978,7 @@ CONFIG_HW_RANDOM_AMD=m
 # CONFIG_HW_RANDOM_BA431 is not set
 CONFIG_HW_RANDOM_VIA=m
 CONFIG_HW_RANDOM_VIRTIO=y
-CONFIG_HW_RANDOM_XIPHERA=m
+# CONFIG_HW_RANDOM_XIPHERA is not set
 # CONFIG_APPLICOM is not set
 # CONFIG_MWAVE is not set
 CONFIG_DEVMEM=y
@@ -3078,13 +3050,13 @@ CONFIG_I2C_ALGOBIT=m
 # CONFIG_I2C_ALI15X3 is not set
 # CONFIG_I2C_AMD756 is not set
 # CONFIG_I2C_AMD8111 is not set
-CONFIG_I2C_AMD_MP2=m
+# CONFIG_I2C_AMD_MP2 is not set
 CONFIG_I2C_I801=m
 CONFIG_I2C_ISCH=m
 CONFIG_I2C_ISMT=m
 # CONFIG_I2C_PIIX4 is not set
 # CONFIG_I2C_NFORCE2 is not set
-CONFIG_I2C_NVIDIA_GPU=m
+# CONFIG_I2C_NVIDIA_GPU is not set
 # CONFIG_I2C_SIS5595 is not set
 # CONFIG_I2C_SIS630 is not set
 # CONFIG_I2C_SIS96X is not set
@@ -3164,7 +3136,7 @@ CONFIG_SPI_BITBANG=m
 #
 # SPI Multiplexer support
 #
-CONFIG_SPI_MUX=m
+# CONFIG_SPI_MUX is not set
 
 #
 # SPI Protocol Masters
@@ -3197,9 +3169,9 @@ CONFIG_PTP_1588_CLOCK=y
 CONFIG_DP83640_PHY=y
 # CONFIG_PTP_1588_CLOCK_INES is not set
 CONFIG_PTP_1588_CLOCK_KVM=y
-CONFIG_PTP_1588_CLOCK_IDT82P33=m
-CONFIG_PTP_1588_CLOCK_IDTCM=m
-CONFIG_PTP_1588_CLOCK_VMW=m
+# CONFIG_PTP_1588_CLOCK_IDT82P33 is not set
+# CONFIG_PTP_1588_CLOCK_IDTCM is not set
+# CONFIG_PTP_1588_CLOCK_VMW is not set
 # end of PTP clock support
 
 CONFIG_PINCTRL=y
@@ -3212,7 +3184,7 @@ CONFIG_PINCTRL_AMD=m
 # CONFIG_PINCTRL_SX150X is not set
 CONFIG_PINCTRL_BAYTRAIL=y
 # CONFIG_PINCTRL_CHERRYVIEW is not set
-CONFIG_PINCTRL_LYNXPOINT=m
+# CONFIG_PINCTRL_LYNXPOINT is not set
 CONFIG_PINCTRL_INTEL=y
 CONFIG_PINCTRL_BROXTON=m
 CONFIG_PINCTRL_CANNONLAKE=m
@@ -3252,7 +3224,7 @@ CONFIG_GPIO_ICH=m
 # CONFIG_GPIO_MB86S7X is not set
 # CONFIG_GPIO_VX855 is not set
 # CONFIG_GPIO_XILINX is not set
-CONFIG_GPIO_AMD_FCH=m
+# CONFIG_GPIO_AMD_FCH is not set
 # end of Memory mapped GPIO drivers
 
 #
@@ -3273,7 +3245,7 @@ CONFIG_GPIO_AMD_FCH=m
 # CONFIG_GPIO_MAX7300 is not set
 # CONFIG_GPIO_MAX732X is not set
 # CONFIG_GPIO_PCA953X is not set
-CONFIG_GPIO_PCA9570=m
+# CONFIG_GPIO_PCA9570 is not set
 # CONFIG_GPIO_PCF857X is not set
 # CONFIG_GPIO_TPIC2810 is not set
 # end of I2C GPIO expanders
@@ -3281,7 +3253,6 @@ CONFIG_GPIO_PCA9570=m
 #
 # MFD GPIO expanders
 #
-CONFIG_GPIO_MSIC=y
 # end of MFD GPIO expanders
 
 #
@@ -3310,7 +3281,7 @@ CONFIG_GPIO_MSIC=y
 #
 # end of USB GPIO expanders
 
-CONFIG_GPIO_AGGREGATOR=m
+# CONFIG_GPIO_AGGREGATOR is not set
 # CONFIG_GPIO_MOCKUP is not set
 # CONFIG_W1 is not set
 CONFIG_POWER_RESET=y
@@ -3322,7 +3293,7 @@ CONFIG_POWER_SUPPLY_HWMON=y
 # CONFIG_GENERIC_ADC_BATTERY is not set
 # CONFIG_TEST_POWER is not set
 # CONFIG_CHARGER_ADP5061 is not set
-CONFIG_BATTERY_CW2015=m
+# CONFIG_BATTERY_CW2015 is not set
 # CONFIG_BATTERY_DS2780 is not set
 # CONFIG_BATTERY_DS2781 is not set
 # CONFIG_BATTERY_DS2782 is not set
@@ -3339,14 +3310,14 @@ CONFIG_BATTERY_CW2015=m
 # CONFIG_CHARGER_BQ2415X is not set
 # CONFIG_CHARGER_BQ24257 is not set
 # CONFIG_CHARGER_BQ24735 is not set
-CONFIG_CHARGER_BQ2515X=m
+# CONFIG_CHARGER_BQ2515X is not set
 # CONFIG_CHARGER_BQ25890 is not set
 # CONFIG_CHARGER_BQ25980 is not set
 # CONFIG_CHARGER_SMB347 is not set
 # CONFIG_BATTERY_GAUGE_LTC2941 is not set
 # CONFIG_BATTERY_RT5033 is not set
 # CONFIG_CHARGER_RT9455 is not set
-CONFIG_CHARGER_BD99954=m
+# CONFIG_CHARGER_BD99954 is not set
 CONFIG_HWMON=y
 # CONFIG_HWMON_DEBUG_CHIP is not set
 
@@ -3373,17 +3344,17 @@ CONFIG_HWMON=y
 # CONFIG_SENSORS_ADT7475 is not set
 # CONFIG_SENSORS_AS370 is not set
 # CONFIG_SENSORS_ASC7621 is not set
-CONFIG_SENSORS_AXI_FAN_CONTROL=m
+# CONFIG_SENSORS_AXI_FAN_CONTROL is not set
 # CONFIG_SENSORS_K8TEMP is not set
 # CONFIG_SENSORS_K10TEMP is not set
 # CONFIG_SENSORS_FAM15H_POWER is not set
-CONFIG_SENSORS_AMD_ENERGY=m
+# CONFIG_SENSORS_AMD_ENERGY is not set
 # CONFIG_SENSORS_APPLESMC is not set
 # CONFIG_SENSORS_ASB100 is not set
 # CONFIG_SENSORS_ASPEED is not set
 # CONFIG_SENSORS_ATXP1 is not set
-CONFIG_SENSORS_CORSAIR_CPRO=m
-CONFIG_SENSORS_DRIVETEMP=m
+# CONFIG_SENSORS_CORSAIR_CPRO is not set
+# CONFIG_SENSORS_DRIVETEMP is not set
 # CONFIG_SENSORS_DS620 is not set
 # CONFIG_SENSORS_DS1621 is not set
 CONFIG_SENSORS_DELL_SMM=m
@@ -3408,9 +3379,8 @@ CONFIG_SENSORS_JC42=m
 # CONFIG_SENSORS_POWR1220 is not set
 # CONFIG_SENSORS_LINEAGE is not set
 # CONFIG_SENSORS_LTC2945 is not set
-CONFIG_SENSORS_LTC2947=m
-CONFIG_SENSORS_LTC2947_I2C=m
-CONFIG_SENSORS_LTC2947_SPI=m
+# CONFIG_SENSORS_LTC2947_I2C is not set
+# CONFIG_SENSORS_LTC2947_SPI is not set
 # CONFIG_SENSORS_LTC2990 is not set
 # CONFIG_SENSORS_LTC4151 is not set
 # CONFIG_SENSORS_LTC4215 is not set
@@ -3432,9 +3402,9 @@ CONFIG_SENSORS_LTC2947_SPI=m
 # CONFIG_SENSORS_MAX6697 is not set
 # CONFIG_SENSORS_MAX31790 is not set
 # CONFIG_SENSORS_MCP3021 is not set
-CONFIG_SENSORS_MLXREG_FAN=m
+# CONFIG_SENSORS_MLXREG_FAN is not set
 # CONFIG_SENSORS_TC654 is not set
-CONFIG_SENSORS_MR75203=m
+# CONFIG_SENSORS_MR75203 is not set
 # CONFIG_SENSORS_ADCXX is not set
 # CONFIG_SENSORS_LM63 is not set
 # CONFIG_SENSORS_LM70 is not set
@@ -3459,13 +3429,13 @@ CONFIG_SENSORS_MR75203=m
 # CONFIG_SENSORS_NCT6775 is not set
 # CONFIG_SENSORS_NCT7802 is not set
 # CONFIG_SENSORS_NCT7904 is not set
-CONFIG_SENSORS_NPCM7XX=m
+# CONFIG_SENSORS_NPCM7XX is not set
 # CONFIG_SENSORS_PCF8591 is not set
 CONFIG_PMBUS=m
 # CONFIG_SENSORS_PMBUS is not set
-CONFIG_SENSORS_ADM1266=m
+# CONFIG_SENSORS_ADM1266 is not set
 # CONFIG_SENSORS_ADM1275 is not set
-CONFIG_SENSORS_BEL_PFE=m
+# CONFIG_SENSORS_BEL_PFE is not set
 # CONFIG_SENSORS_IBM_CFFPS is not set
 # CONFIG_SENSORS_INSPUR_IPSPS is not set
 # CONFIG_SENSORS_IR35221 is not set
@@ -3482,7 +3452,7 @@ CONFIG_SENSORS_BEL_PFE=m
 # CONFIG_SENSORS_MAX31785 is not set
 # CONFIG_SENSORS_MAX34440 is not set
 # CONFIG_SENSORS_MAX8688 is not set
-CONFIG_SENSORS_MP2975=m
+# CONFIG_SENSORS_MP2975 is not set
 # CONFIG_SENSORS_PXE1610 is not set
 # CONFIG_SENSORS_TPS40422 is not set
 # CONFIG_SENSORS_TPS53679 is not set
@@ -3520,7 +3490,7 @@ CONFIG_SENSORS_SHT15=m
 # CONFIG_SENSORS_TMP108 is not set
 # CONFIG_SENSORS_TMP401 is not set
 # CONFIG_SENSORS_TMP421 is not set
-CONFIG_SENSORS_TMP513=m
+# CONFIG_SENSORS_TMP513 is not set
 # CONFIG_SENSORS_VIA_CPUTEMP is not set
 # CONFIG_SENSORS_VIA686A is not set
 # CONFIG_SENSORS_VT1211 is not set
@@ -3536,7 +3506,6 @@ CONFIG_SENSORS_TMP513=m
 # CONFIG_SENSORS_W83627HF is not set
 # CONFIG_SENSORS_W83627EHF is not set
 # CONFIG_SENSORS_XGENE is not set
-CONFIG_SENSORS_INTEL_M10_BMC_HWMON=m
 
 #
 # ACPI drivers
@@ -3544,7 +3513,7 @@ CONFIG_SENSORS_INTEL_M10_BMC_HWMON=m
 CONFIG_SENSORS_ACPI_POWER=m
 # CONFIG_SENSORS_ATK0110 is not set
 CONFIG_THERMAL=y
-CONFIG_THERMAL_NETLINK=y
+# CONFIG_THERMAL_NETLINK is not set
 # CONFIG_THERMAL_STATISTICS is not set
 CONFIG_THERMAL_EMERGENCY_POWEROFF_DELAY_MS=0
 CONFIG_THERMAL_HWMON=y
@@ -3556,7 +3525,6 @@ CONFIG_THERMAL_GOV_FAIR_SHARE=y
 CONFIG_THERMAL_GOV_STEP_WISE=y
 # CONFIG_THERMAL_GOV_BANG_BANG is not set
 CONFIG_THERMAL_GOV_USER_SPACE=y
-# CONFIG_THERMAL_GOV_POWER_ALLOCATOR is not set
 # CONFIG_THERMAL_EMULATION is not set
 
 #
@@ -3598,7 +3566,7 @@ CONFIG_SOFT_WATCHDOG=y
 CONFIG_WDAT_WDT=m
 # CONFIG_XILINX_WATCHDOG is not set
 # CONFIG_ZIIRAVE_WATCHDOG is not set
-CONFIG_MLX_WDT=m
+# CONFIG_MLX_WDT is not set
 # CONFIG_CADENCE_WATCHDOG is not set
 # CONFIG_DW_WATCHDOG is not set
 # CONFIG_MAX63XX_WATCHDOG is not set
@@ -3629,7 +3597,7 @@ CONFIG_HPWDT_NMI_DECODING=y
 # CONFIG_CPU5_WDT is not set
 # CONFIG_SMSC_SCH311X_WDT is not set
 # CONFIG_SMSC37B787_WDT is not set
-CONFIG_TQMX86_WDT=m
+# CONFIG_TQMX86_WDT is not set
 # CONFIG_VIA_WDT is not set
 # CONFIG_W83627HF_WDT is not set
 # CONFIG_W83877F_WDT is not set
@@ -3683,14 +3651,11 @@ CONFIG_MFD_CORE=y
 # CONFIG_MFD_INTEL_QUARK_I2C_GPIO is not set
 CONFIG_LPC_ICH=m
 CONFIG_LPC_SCH=m
-# CONFIG_INTEL_SOC_PMIC_BXTWC is not set
 # CONFIG_INTEL_SOC_PMIC_CHTDC_TI is not set
-# CONFIG_INTEL_SOC_PMIC_MRFLD is not set
 CONFIG_MFD_INTEL_LPSS=y
 CONFIG_MFD_INTEL_LPSS_ACPI=y
 CONFIG_MFD_INTEL_LPSS_PCI=y
-CONFIG_MFD_INTEL_MSIC=y
-CONFIG_MFD_INTEL_PMC_BXT=m
+# CONFIG_MFD_INTEL_PMC_BXT is not set
 # CONFIG_MFD_IQS62X is not set
 # CONFIG_MFD_JANZ_CMODIO is not set
 # CONFIG_MFD_KEMPLD is not set
@@ -3749,18 +3714,16 @@ CONFIG_MFD_SYSCON=y
 # CONFIG_MFD_WM831X_SPI is not set
 # CONFIG_MFD_WM8350_I2C is not set
 # CONFIG_MFD_WM8994 is not set
-# CONFIG_RAVE_SP_CORE is not set
-CONFIG_MFD_INTEL_M10_BMC=m
+# CONFIG_MFD_INTEL_M10_BMC is not set
 # end of Multifunction device drivers
 
 # CONFIG_REGULATOR is not set
 # CONFIG_RC_CORE is not set
 CONFIG_CEC_CORE=m
-CONFIG_CEC_NOTIFIER=y
 CONFIG_MEDIA_CEC_SUPPORT=y
-CONFIG_CEC_CH7322=m
+# CONFIG_CEC_CH7322 is not set
 # CONFIG_CEC_GPIO is not set
-CONFIG_CEC_SECO=m
+# CONFIG_CEC_SECO is not set
 CONFIG_USB_PULSE8_CEC=m
 CONFIG_USB_RAINSHADOW_CEC=m
 # CONFIG_MEDIA_SUPPORT is not set
@@ -3771,10 +3734,9 @@ CONFIG_USB_RAINSHADOW_CEC=m
 # CONFIG_AGP is not set
 CONFIG_INTEL_GTT=m
 CONFIG_VGA_ARB=y
-CONFIG_VGA_ARB_MAX_GPUS=16
+CONFIG_VGA_ARB_MAX_GPUS=64
 # CONFIG_VGA_SWITCHEROO is not set
 CONFIG_DRM=m
-CONFIG_DRM_MIPI_DBI=m
 CONFIG_DRM_MIPI_DSI=y
 CONFIG_DRM_DP_AUX_CHARDEV=y
 # CONFIG_DRM_DEBUG_SELFTEST is not set
@@ -3786,10 +3748,6 @@ CONFIG_DRM_FBDEV_OVERALLOC=100
 CONFIG_DRM_DP_CEC=y
 CONFIG_DRM_TTM=m
 CONFIG_DRM_TTM_DMA_PAGE_POOL=y
-CONFIG_DRM_VRAM_HELPER=m
-CONFIG_DRM_TTM_HELPER=m
-CONFIG_DRM_GEM_CMA_HELPER=y
-CONFIG_DRM_KMS_CMA_HELPER=y
 CONFIG_DRM_GEM_SHMEM_HELPER=y
 CONFIG_DRM_SCHED=m
 
@@ -3825,9 +3783,9 @@ CONFIG_DRM_AMDGPU=m
 #
 CONFIG_DRM_AMD_DC=y
 CONFIG_DRM_AMD_DC_DCN=y
-CONFIG_DRM_AMD_DC_DCN3_0=y
-CONFIG_DRM_AMD_DC_HDCP=y
-CONFIG_DRM_AMD_DC_SI=y
+# CONFIG_DRM_AMD_DC_DCN3_0 is not set
+# CONFIG_DRM_AMD_DC_HDCP is not set
+# CONFIG_DRM_AMD_DC_SI is not set
 # end of Display Engine Configuration
 
 CONFIG_HSA_AMD=y
@@ -3875,16 +3833,16 @@ CONFIG_DRM_PANEL_BRIDGE=y
 
 # CONFIG_DRM_ETNAVIV is not set
 # CONFIG_DRM_CIRRUS_QEMU is not set
-CONFIG_DRM_GM12U320=m
+# CONFIG_DRM_GM12U320 is not set
 # CONFIG_TINYDRM_HX8357D is not set
 # CONFIG_TINYDRM_ILI9225 is not set
 # CONFIG_TINYDRM_ILI9341 is not set
-CONFIG_TINYDRM_ILI9486=m
+# CONFIG_TINYDRM_ILI9486 is not set
 # CONFIG_TINYDRM_MI0283QT is not set
 # CONFIG_TINYDRM_REPAPER is not set
 # CONFIG_TINYDRM_ST7586 is not set
 # CONFIG_TINYDRM_ST7735R is not set
-CONFIG_DRM_VBOXVIDEO=m
+# CONFIG_DRM_VBOXVIDEO is not set
 # CONFIG_DRM_LEGACY is not set
 CONFIG_DRM_PANEL_ORIENTATION_QUIRKS=y
 
@@ -3974,7 +3932,7 @@ CONFIG_LCD_CLASS_DEVICE=m
 # CONFIG_LCD_HX8357 is not set
 # CONFIG_LCD_OTM3225A is not set
 CONFIG_BACKLIGHT_CLASS_DEVICE=y
-CONFIG_BACKLIGHT_KTD253=m
+# CONFIG_BACKLIGHT_KTD253 is not set
 # CONFIG_BACKLIGHT_PWM is not set
 # CONFIG_BACKLIGHT_APPLE is not set
 # CONFIG_BACKLIGHT_QCOM_WLED is not set
@@ -4002,7 +3960,7 @@ CONFIG_DUMMY_CONSOLE_ROWS=25
 CONFIG_FRAMEBUFFER_CONSOLE=y
 CONFIG_FRAMEBUFFER_CONSOLE_DETECT_PRIMARY=y
 # CONFIG_FRAMEBUFFER_CONSOLE_ROTATION is not set
-CONFIG_FRAMEBUFFER_CONSOLE_DEFERRED_TAKEOVER=y
+# CONFIG_FRAMEBUFFER_CONSOLE_DEFERRED_TAKEOVER is not set
 # end of Console display driver support
 
 # CONFIG_LOGO is not set
@@ -4031,14 +3989,14 @@ CONFIG_HID_ASUS=m
 # CONFIG_HID_AUREAL is not set
 CONFIG_HID_BELKIN=m
 # CONFIG_HID_BETOP_FF is not set
-CONFIG_HID_BIGBEN_FF=m
+# CONFIG_HID_BIGBEN_FF is not set
 CONFIG_HID_CHERRY=m
 CONFIG_HID_CHICONY=m
 CONFIG_HID_CORSAIR=m
-CONFIG_HID_COUGAR=m
-CONFIG_HID_MACALLY=m
+# CONFIG_HID_COUGAR is not set
+# CONFIG_HID_MACALLY is not set
 CONFIG_HID_CMEDIA=m
-CONFIG_HID_CREATIVE_SB0540=m
+# CONFIG_HID_CREATIVE_SB0540 is not set
 CONFIG_HID_CYPRESS=m
 # CONFIG_HID_DRAGONRISE is not set
 # CONFIG_HID_EMS_FF is not set
@@ -4048,15 +4006,15 @@ CONFIG_HID_ELO=m
 CONFIG_HID_EZKEY=m
 CONFIG_HID_GEMBIRD=m
 CONFIG_HID_GFRM=m
-CONFIG_HID_GLORIOUS=m
+# CONFIG_HID_GLORIOUS is not set
 # CONFIG_HID_HOLTEK is not set
-CONFIG_HID_VIVALDI=m
+# CONFIG_HID_VIVALDI is not set
 CONFIG_HID_GT683R=m
 # CONFIG_HID_KEYTOUCH is not set
 # CONFIG_HID_KYE is not set
 # CONFIG_HID_UCLOGIC is not set
 # CONFIG_HID_WALTOP is not set
-CONFIG_HID_VIEWSONIC=m
+# CONFIG_HID_VIEWSONIC is not set
 # CONFIG_HID_GYRATION is not set
 # CONFIG_HID_ICADE is not set
 CONFIG_HID_ITE=m
@@ -4073,7 +4031,7 @@ CONFIG_HID_LOGITECH_HIDPP=m
 # CONFIG_LOGIG940_FF is not set
 # CONFIG_LOGIWHEELS_FF is not set
 # CONFIG_HID_MAGICMOUSE is not set
-CONFIG_HID_MALTRON=m
+# CONFIG_HID_MALTRON is not set
 # CONFIG_HID_MAYFLASH is not set
 # CONFIG_HID_REDRAGON is not set
 CONFIG_HID_MICROSOFT=m
@@ -4106,7 +4064,7 @@ CONFIG_HID_HYPERV_MOUSE=m
 # CONFIG_HID_THINGM is not set
 # CONFIG_HID_THRUSTMASTER is not set
 # CONFIG_HID_UDRAW_PS3 is not set
-CONFIG_HID_U2FZERO=m
+# CONFIG_HID_U2FZERO is not set
 # CONFIG_HID_WACOM is not set
 # CONFIG_HID_WIIMOTE is not set
 CONFIG_HID_XINMO=m
@@ -4115,7 +4073,7 @@ CONFIG_HID_XINMO=m
 CONFIG_HID_SENSOR_HUB=y
 CONFIG_HID_SENSOR_CUSTOM_SENSOR=m
 CONFIG_HID_ALPS=m
-CONFIG_HID_MCP2221=m
+# CONFIG_HID_MCP2221 is not set
 # end of Special HID drivers
 
 #
@@ -4136,7 +4094,7 @@ CONFIG_I2C_HID=m
 # Intel ISH HID support
 #
 CONFIG_INTEL_ISH_HID=m
-CONFIG_INTEL_ISH_FIRMWARE_DOWNLOADER=m
+# CONFIG_INTEL_ISH_FIRMWARE_DOWNLOADER is not set
 # end of Intel ISH HID support
 # end of HID support
 
@@ -4170,7 +4128,7 @@ CONFIG_USB_AUTOSUSPEND_DELAY=2
 CONFIG_USB_XHCI_HCD=y
 CONFIG_USB_XHCI_DBGCAP=y
 CONFIG_USB_XHCI_PCI=y
-CONFIG_USB_XHCI_PCI_RENESAS=y
+# CONFIG_USB_XHCI_PCI_RENESAS is not set
 # CONFIG_USB_XHCI_PLATFORM is not set
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_EHCI_ROOT_HUB_TT=y
@@ -4305,7 +4263,7 @@ CONFIG_USB_SERIAL_UPD78F0730=m
 # CONFIG_USB_IDMOUSE is not set
 # CONFIG_USB_FTDI_ELAN is not set
 # CONFIG_USB_APPLEDISPLAY is not set
-CONFIG_APPLE_MFI_FASTCHARGE=m
+# CONFIG_APPLE_MFI_FASTCHARGE is not set
 # CONFIG_USB_SISUSBVGA is not set
 # CONFIG_USB_LD is not set
 # CONFIG_USB_TRANCEVIBRATOR is not set
@@ -4335,27 +4293,26 @@ CONFIG_TYPEC=y
 CONFIG_TYPEC_TCPM=y
 CONFIG_TYPEC_TCPCI=y
 CONFIG_TYPEC_RT1711H=y
-CONFIG_TYPEC_TCPCI_MAXIM=m
+# CONFIG_TYPEC_TCPCI_MAXIM is not set
 CONFIG_TYPEC_FUSB302=m
 CONFIG_TYPEC_UCSI=y
-CONFIG_UCSI_CCG=m
+# CONFIG_UCSI_CCG is not set
 CONFIG_UCSI_ACPI=y
-CONFIG_TYPEC_HD3SS3220=m
+# CONFIG_TYPEC_HD3SS3220 is not set
 CONFIG_TYPEC_TPS6598X=m
-CONFIG_TYPEC_STUSB160X=m
+# CONFIG_TYPEC_STUSB160X is not set
 
 #
 # USB Type-C Multiplexer/DeMultiplexer Switch support
 #
 CONFIG_TYPEC_MUX_PI3USB30532=m
-CONFIG_TYPEC_MUX_INTEL_PMC=m
 # end of USB Type-C Multiplexer/DeMultiplexer Switch support
 
 #
 # USB Type-C Alternate Mode drivers
 #
 CONFIG_TYPEC_DP_ALTMODE=y
-CONFIG_TYPEC_NVIDIA_ALTMODE=m
+# CONFIG_TYPEC_NVIDIA_ALTMODE is not set
 # end of USB Type-C Alternate Mode drivers
 
 CONFIG_USB_ROLE_SWITCH=y
@@ -4378,7 +4335,6 @@ CONFIG_MMC_SDHCI_ACPI=m
 CONFIG_MMC_SDHCI_PLTFM=m
 # CONFIG_MMC_SDHCI_F_SDH30 is not set
 # CONFIG_MMC_WBSD is not set
-CONFIG_MMC_ALCOR=m
 # CONFIG_MMC_TIFM_SD is not set
 # CONFIG_MMC_SPI is not set
 # CONFIG_MMC_CB710 is not set
@@ -4389,7 +4345,7 @@ CONFIG_MMC_ALCOR=m
 CONFIG_MMC_REALTEK_PCI=m
 CONFIG_MMC_REALTEK_USB=m
 CONFIG_MMC_CQHCI=m
-CONFIG_MMC_HSQ=m
+# CONFIG_MMC_HSQ is not set
 # CONFIG_MMC_TOSHIBA_PCI is not set
 # CONFIG_MMC_MTK is not set
 # CONFIG_MMC_SDHCI_XENON is not set
@@ -4397,7 +4353,7 @@ CONFIG_MMC_HSQ=m
 CONFIG_NEW_LEDS=y
 CONFIG_LEDS_CLASS=y
 # CONFIG_LEDS_CLASS_FLASH is not set
-CONFIG_LEDS_CLASS_MULTICOLOR=m
+# CONFIG_LEDS_CLASS_MULTICOLOR is not set
 # CONFIG_LEDS_BRIGHTNESS_HW_CHANGED is not set
 
 #
@@ -4405,13 +4361,13 @@ CONFIG_LEDS_CLASS_MULTICOLOR=m
 #
 # CONFIG_LEDS_APU is not set
 # CONFIG_LEDS_LM3530 is not set
-CONFIG_LEDS_LM3532=m
+# CONFIG_LEDS_LM3532 is not set
 # CONFIG_LEDS_LM3642 is not set
 # CONFIG_LEDS_PCA9532 is not set
 # CONFIG_LEDS_GPIO is not set
 # CONFIG_LEDS_LP3944 is not set
 # CONFIG_LEDS_LP3952 is not set
-CONFIG_LEDS_LP50XX=m
+# CONFIG_LEDS_LP50XX is not set
 # CONFIG_LEDS_CLEVO_MAIL is not set
 # CONFIG_LEDS_PCA955X is not set
 # CONFIG_LEDS_PCA963X is not set
@@ -4456,17 +4412,16 @@ CONFIG_LEDS_TRIGGER_DEFAULT_ON=m
 # CONFIG_LEDS_TRIGGER_PANIC is not set
 # CONFIG_LEDS_TRIGGER_NETDEV is not set
 # CONFIG_LEDS_TRIGGER_PATTERN is not set
-# CONFIG_LEDS_TRIGGER_AUDIO is not set
+CONFIG_LEDS_TRIGGER_AUDIO=m
 # CONFIG_ACCESSIBILITY is not set
 CONFIG_INFINIBAND=m
 # CONFIG_INFINIBAND_USER_MAD is not set
 # CONFIG_INFINIBAND_USER_ACCESS is not set
 # CONFIG_INFINIBAND_ADDR_TRANS is not set
 CONFIG_INFINIBAND_VIRT_DMA=y
-CONFIG_INFINIBAND_MTHCA=m
-CONFIG_INFINIBAND_MTHCA_DEBUG=y
+# CONFIG_INFINIBAND_MTHCA is not set
 # CONFIG_MLX4_INFINIBAND is not set
-CONFIG_INFINIBAND_OCRDMA=m
+# CONFIG_INFINIBAND_OCRDMA is not set
 # CONFIG_INFINIBAND_VMWARE_PVRDMA is not set
 # CONFIG_INFINIBAND_BNXT_RE is not set
 # CONFIG_INFINIBAND_QEDR is not set
@@ -4500,7 +4455,7 @@ CONFIG_RTC_INTF_DEV=y
 # I2C RTC drivers
 #
 # CONFIG_RTC_DRV_ABB5ZES3 is not set
-CONFIG_RTC_DRV_ABEOZ9=m
+# CONFIG_RTC_DRV_ABEOZ9 is not set
 # CONFIG_RTC_DRV_ABX80X is not set
 CONFIG_RTC_DRV_DS1307=m
 # CONFIG_RTC_DRV_DS1307_CENTURY is not set
@@ -4526,10 +4481,10 @@ CONFIG_RTC_DRV_FM3130=m
 CONFIG_RTC_DRV_RX8581=m
 CONFIG_RTC_DRV_RX8025=m
 CONFIG_RTC_DRV_EM3027=m
-CONFIG_RTC_DRV_RV3028=m
-CONFIG_RTC_DRV_RV3032=m
+# CONFIG_RTC_DRV_RV3028 is not set
+# CONFIG_RTC_DRV_RV3032 is not set
 # CONFIG_RTC_DRV_RV8803 is not set
-CONFIG_RTC_DRV_SD3078=m
+# CONFIG_RTC_DRV_SD3078 is not set
 
 #
 # SPI RTC drivers
@@ -4599,7 +4554,7 @@ CONFIG_DMA_VIRTUAL_CHANNELS=y
 CONFIG_DMA_ACPI=y
 # CONFIG_ALTERA_MSGDMA is not set
 CONFIG_INTEL_IDMA64=m
-CONFIG_INTEL_IDXD=m
+# CONFIG_INTEL_IDXD is not set
 CONFIG_INTEL_IOATDMA=m
 # CONFIG_PLX_DMA is not set
 # CONFIG_QCOM_HIDMA_MGMT is not set
@@ -4607,8 +4562,8 @@ CONFIG_INTEL_IOATDMA=m
 CONFIG_DW_DMAC_CORE=y
 CONFIG_DW_DMAC=m
 CONFIG_DW_DMAC_PCI=y
-CONFIG_DW_EDMA=m
-CONFIG_DW_EDMA_PCIE=m
+# CONFIG_DW_EDMA is not set
+# CONFIG_DW_EDMA_PCIE is not set
 CONFIG_HSU_DMA=y
 # CONFIG_SF_PDMA is not set
 
@@ -4627,8 +4582,7 @@ CONFIG_SYNC_FILE=y
 # CONFIG_UDMABUF is not set
 # CONFIG_DMABUF_MOVE_NOTIFY is not set
 # CONFIG_DMABUF_SELFTESTS is not set
-CONFIG_DMABUF_HEAPS=y
-CONFIG_DMABUF_HEAPS_SYSTEM=y
+# CONFIG_DMABUF_HEAPS is not set
 # end of DMABUF options
 
 CONFIG_DCA=m
@@ -4661,24 +4615,19 @@ CONFIG_VIRTIO=y
 CONFIG_VIRTIO_MENU=y
 CONFIG_VIRTIO_PCI=y
 CONFIG_VIRTIO_PCI_LEGACY=y
-CONFIG_VIRTIO_VDPA=m
 # CONFIG_VIRTIO_PMEM is not set
 CONFIG_VIRTIO_BALLOON=m
 CONFIG_VIRTIO_INPUT=m
 CONFIG_VIRTIO_MMIO=y
 # CONFIG_VIRTIO_MMIO_CMDLINE_DEVICES is not set
 CONFIG_VIRTIO_DMA_SHARED_BUFFER=m
-CONFIG_VDPA=m
-CONFIG_VDPA_SIM=m
-CONFIG_IFCVF=m
+# CONFIG_VDPA is not set
 CONFIG_VHOST_IOTLB=m
-CONFIG_VHOST_RING=m
 CONFIG_VHOST=m
 CONFIG_VHOST_MENU=y
 CONFIG_VHOST_NET=m
 # CONFIG_VHOST_SCSI is not set
 CONFIG_VHOST_VSOCK=m
-CONFIG_VHOST_VDPA=m
 # CONFIG_VHOST_CROSS_ENDIAN_LEGACY is not set
 
 #
@@ -4702,7 +4651,7 @@ CONFIG_X86_PLATFORM_DEVICES=y
 # CONFIG_EEEPC_LAPTOP is not set
 # CONFIG_DCDBAS is not set
 CONFIG_DELL_SMBIOS=m
-# CONFIG_DELL_LAPTOP is not set
+CONFIG_DELL_LAPTOP=m
 CONFIG_DELL_RBTN=m
 # CONFIG_DELL_RBU is not set
 CONFIG_DELL_SMO8800=m
@@ -4716,13 +4665,13 @@ CONFIG_DELL_SMO8800=m
 # CONFIG_IDEAPAD_LAPTOP is not set
 # CONFIG_SENSORS_HDAPS is not set
 # CONFIG_THINKPAD_ACPI is not set
-CONFIG_INTEL_ATOMISP2_PM=m
+# CONFIG_INTEL_ATOMISP2_PM is not set
 # CONFIG_INTEL_HID_EVENT is not set
 # CONFIG_INTEL_INT0002_VGPIO is not set
 # CONFIG_INTEL_MENLOW is not set
 # CONFIG_INTEL_OAKTRAIL is not set
 # CONFIG_INTEL_VBTN is not set
-CONFIG_SURFACE_3_POWER_OPREGION=m
+# CONFIG_SURFACE_3_POWER_OPREGION is not set
 # CONFIG_SURFACE_PRO3_BUTTON is not set
 # CONFIG_MSI_LAPTOP is not set
 # CONFIG_PCENGINES_APU2 is not set
@@ -4734,9 +4683,9 @@ CONFIG_SURFACE_3_POWER_OPREGION=m
 # CONFIG_COMPAL_LAPTOP is not set
 # CONFIG_PANASONIC_LAPTOP is not set
 # CONFIG_SONY_LAPTOP is not set
-CONFIG_SYSTEM76_ACPI=m
+# CONFIG_SYSTEM76_ACPI is not set
 # CONFIG_TOPSTAR_LAPTOP is not set
-CONFIG_I2C_MULTI_INSTANTIATE=m
+# CONFIG_I2C_MULTI_INSTANTIATE is not set
 CONFIG_MLX_PLATFORM=m
 CONFIG_INTEL_IPS=m
 CONFIG_INTEL_RST=m
@@ -4745,25 +4694,20 @@ CONFIG_INTEL_RST=m
 #
 # Intel Speed Select Technology interface support
 #
-CONFIG_INTEL_SPEED_SELECT_INTERFACE=m
+# CONFIG_INTEL_SPEED_SELECT_INTERFACE is not set
 # end of Intel Speed Select Technology interface support
 
 CONFIG_INTEL_TURBO_MAX_3=y
-CONFIG_INTEL_UNCORE_FREQ_CONTROL=m
-CONFIG_INTEL_MFLD_THERMAL=m
-CONFIG_INTEL_MID_POWER_BUTTON=m
-CONFIG_INTEL_PMC_CORE=y
+# CONFIG_INTEL_UNCORE_FREQ_CONTROL is not set
+CONFIG_INTEL_PMC_CORE=m
 # CONFIG_INTEL_PUNIT_IPC is not set
-CONFIG_INTEL_SCU_IPC=y
-CONFIG_INTEL_SCU=y
-CONFIG_INTEL_SCU_PCI=y
-CONFIG_INTEL_SCU_PLATFORM=m
-CONFIG_INTEL_SCU_IPC_UTIL=m
+# CONFIG_INTEL_SCU_PCI is not set
+# CONFIG_INTEL_SCU_PLATFORM is not set
 CONFIG_PMC_ATOM=y
 # CONFIG_CHROME_PLATFORMS is not set
 CONFIG_MELLANOX_PLATFORM=y
 CONFIG_MLXREG_HOTPLUG=m
-CONFIG_MLXREG_IO=m
+# CONFIG_MLXREG_IO is not set
 CONFIG_HAVE_CLK=y
 CONFIG_CLKDEV_LOOKUP=y
 CONFIG_HAVE_CLK_PREPARE=y
@@ -4808,7 +4752,7 @@ CONFIG_INTEL_IOMMU=y
 # CONFIG_INTEL_IOMMU_SVM is not set
 CONFIG_INTEL_IOMMU_DEFAULT_ON=y
 CONFIG_INTEL_IOMMU_FLOPPY_WA=y
-CONFIG_INTEL_IOMMU_SCALABLE_MODE_DEFAULT_ON=y
+# CONFIG_INTEL_IOMMU_SCALABLE_MODE_DEFAULT_ON is not set
 CONFIG_IRQ_REMAP=y
 CONFIG_HYPERV_IOMMU=y
 
@@ -4876,8 +4820,8 @@ CONFIG_HYPERV_IOMMU=y
 CONFIG_IIO=m
 CONFIG_IIO_BUFFER=y
 # CONFIG_IIO_BUFFER_CB is not set
-CONFIG_IIO_BUFFER_DMA=m
-CONFIG_IIO_BUFFER_DMAENGINE=m
+# CONFIG_IIO_BUFFER_DMA is not set
+# CONFIG_IIO_BUFFER_DMAENGINE is not set
 # CONFIG_IIO_BUFFER_HW_CONSUMER is not set
 CONFIG_IIO_KFIFO_BUF=m
 CONFIG_IIO_TRIGGERED_BUFFER=m
@@ -4886,7 +4830,7 @@ CONFIG_IIO_TRIGGER=y
 CONFIG_IIO_CONSUMERS_PER_TRIGGER=2
 # CONFIG_IIO_SW_DEVICE is not set
 # CONFIG_IIO_SW_TRIGGER is not set
-CONFIG_IIO_TRIGGERED_EVENT=m
+# CONFIG_IIO_TRIGGERED_EVENT is not set
 
 #
 # Accelerometers
@@ -4895,9 +4839,8 @@ CONFIG_IIO_TRIGGERED_EVENT=m
 # CONFIG_ADIS16209 is not set
 # CONFIG_ADXL345_I2C is not set
 # CONFIG_ADXL345_SPI is not set
-CONFIG_ADXL372=m
-CONFIG_ADXL372_SPI=m
-CONFIG_ADXL372_I2C=m
+# CONFIG_ADXL372_SPI is not set
+# CONFIG_ADXL372_I2C is not set
 # CONFIG_BMA180 is not set
 # CONFIG_BMA220 is not set
 # CONFIG_BMA400 is not set
@@ -4927,13 +4870,12 @@ CONFIG_HID_SENSOR_ACCEL_3D=m
 #
 # Analog to digital converters
 #
-CONFIG_AD_SIGMA_DELTA=m
 # CONFIG_AD7091R5 is not set
-CONFIG_AD7124=m
+# CONFIG_AD7124 is not set
 # CONFIG_AD7192 is not set
 # CONFIG_AD7266 is not set
 # CONFIG_AD7291 is not set
-CONFIG_AD7292=m
+# CONFIG_AD7292 is not set
 # CONFIG_AD7298 is not set
 # CONFIG_AD7476 is not set
 # CONFIG_AD7606_IFACE_PARALLEL is not set
@@ -4945,7 +4887,7 @@ CONFIG_AD7292=m
 # CONFIG_AD7793 is not set
 # CONFIG_AD7887 is not set
 # CONFIG_AD7923 is not set
-CONFIG_AD7949=m
+# CONFIG_AD7949 is not set
 # CONFIG_AD799X is not set
 # CONFIG_HI8435 is not set
 # CONFIG_HX711 is not set
@@ -4957,12 +4899,12 @@ CONFIG_AD7949=m
 # CONFIG_MAX1027 is not set
 # CONFIG_MAX11100 is not set
 # CONFIG_MAX1118 is not set
-CONFIG_MAX1241=m
+# CONFIG_MAX1241 is not set
 # CONFIG_MAX1363 is not set
 # CONFIG_MAX9611 is not set
 # CONFIG_MCP320X is not set
 # CONFIG_MCP3422 is not set
-CONFIG_MCP3911=m
+# CONFIG_MCP3911 is not set
 # CONFIG_NAU7802 is not set
 # CONFIG_TI_ADC081C is not set
 # CONFIG_TI_ADC0832 is not set
@@ -4986,7 +4928,7 @@ CONFIG_MCP3911=m
 # Amplifiers
 #
 # CONFIG_AD8366 is not set
-CONFIG_HMC425=m
+# CONFIG_HMC425 is not set
 # end of Amplifiers
 
 #
@@ -4994,15 +4936,10 @@ CONFIG_HMC425=m
 #
 # CONFIG_ATLAS_PH_SENSOR is not set
 # CONFIG_ATLAS_EZO_SENSOR is not set
-CONFIG_BME680=m
-CONFIG_BME680_I2C=m
-CONFIG_BME680_SPI=m
+# CONFIG_BME680 is not set
 # CONFIG_CCS811 is not set
 # CONFIG_IAQCORE is not set
-CONFIG_PMS7003=m
-CONFIG_SCD30_CORE=m
-CONFIG_SCD30_I2C=m
-CONFIG_SCD30_SERIAL=m
+# CONFIG_SCD30_CORE is not set
 # CONFIG_SENSIRION_SGP30 is not set
 # CONFIG_SPS30 is not set
 # CONFIG_VZ89X is not set
@@ -5040,12 +4977,12 @@ CONFIG_HID_SENSOR_IIO_TRIGGER=m
 # CONFIG_AD5758 is not set
 # CONFIG_AD5761 is not set
 # CONFIG_AD5764 is not set
-CONFIG_AD5770R=m
+# CONFIG_AD5770R is not set
 # CONFIG_AD5791 is not set
 # CONFIG_AD7303 is not set
 # CONFIG_AD8801 is not set
 # CONFIG_DS4424 is not set
-CONFIG_LTC1660=m
+# CONFIG_LTC1660 is not set
 # CONFIG_LTC2632 is not set
 # CONFIG_M62332 is not set
 # CONFIG_MAX517 is not set
@@ -5053,7 +4990,7 @@ CONFIG_LTC1660=m
 # CONFIG_MCP4922 is not set
 # CONFIG_TI_DAC082S085 is not set
 # CONFIG_TI_DAC5571 is not set
-CONFIG_TI_DAC7311=m
+# CONFIG_TI_DAC7311 is not set
 # CONFIG_TI_DAC7612 is not set
 # end of Digital to analog converters
 
@@ -5087,12 +5024,10 @@ CONFIG_TI_DAC7311=m
 # CONFIG_ADIS16130 is not set
 # CONFIG_ADIS16136 is not set
 # CONFIG_ADIS16260 is not set
-CONFIG_ADXRS290=m
+# CONFIG_ADXRS290 is not set
 # CONFIG_ADXRS450 is not set
 # CONFIG_BMG160 is not set
-CONFIG_FXAS21002C=m
-CONFIG_FXAS21002C_I2C=m
-CONFIG_FXAS21002C_SPI=m
+# CONFIG_FXAS21002C is not set
 CONFIG_HID_SENSOR_GYRO_3D=m
 # CONFIG_MPU3050_I2C is not set
 # CONFIG_IIO_ST_GYRO_3AXIS is not set
@@ -5119,7 +5054,7 @@ CONFIG_HID_SENSOR_GYRO_3D=m
 # CONFIG_AM2315 is not set
 # CONFIG_DHT11 is not set
 # CONFIG_HDC100X is not set
-CONFIG_HDC2010=m
+# CONFIG_HDC2010 is not set
 CONFIG_HID_SENSOR_HUMIDITY=m
 # CONFIG_HTS221 is not set
 # CONFIG_HTU21 is not set
@@ -5132,32 +5067,27 @@ CONFIG_HID_SENSOR_HUMIDITY=m
 #
 # CONFIG_ADIS16400 is not set
 # CONFIG_ADIS16460 is not set
-CONFIG_ADIS16475=m
+# CONFIG_ADIS16475 is not set
 # CONFIG_ADIS16480 is not set
 # CONFIG_BMI160_I2C is not set
 # CONFIG_BMI160_SPI is not set
-CONFIG_FXOS8700=m
-CONFIG_FXOS8700_I2C=m
-CONFIG_FXOS8700_SPI=m
+# CONFIG_FXOS8700_I2C is not set
+# CONFIG_FXOS8700_SPI is not set
 # CONFIG_KMX61 is not set
-CONFIG_INV_ICM42600=m
-CONFIG_INV_ICM42600_I2C=m
-CONFIG_INV_ICM42600_SPI=m
+# CONFIG_INV_ICM42600_I2C is not set
+# CONFIG_INV_ICM42600_SPI is not set
 # CONFIG_INV_MPU6050_I2C is not set
 # CONFIG_INV_MPU6050_SPI is not set
 # CONFIG_IIO_ST_LSM6DSX is not set
 # end of Inertial measurement units
 
-CONFIG_IIO_ADIS_LIB=m
-CONFIG_IIO_ADIS_LIB_BUFFER=y
-
 #
 # Light sensors
 #
 # CONFIG_ACPI_ALS is not set
 # CONFIG_ADJD_S311 is not set
-CONFIG_ADUX1020=m
-CONFIG_AL3010=m
+# CONFIG_ADUX1020 is not set
+# CONFIG_AL3010 is not set
 # CONFIG_AL3320A is not set
 # CONFIG_APDS9300 is not set
 # CONFIG_APDS9960 is not set
@@ -5168,7 +5098,7 @@ CONFIG_AL3010=m
 # CONFIG_CM3232 is not set
 # CONFIG_CM3323 is not set
 # CONFIG_CM36651 is not set
-CONFIG_GP2AP002=m
+# CONFIG_GP2AP002 is not set
 # CONFIG_GP2AP020A00F is not set
 # CONFIG_SENSORS_ISL29018 is not set
 # CONFIG_SENSORS_ISL29028 is not set
@@ -5180,7 +5110,7 @@ CONFIG_HID_SENSOR_PROX=m
 # CONFIG_LTR501 is not set
 # CONFIG_LV0104CS is not set
 # CONFIG_MAX44000 is not set
-CONFIG_MAX44009=m
+# CONFIG_MAX44009 is not set
 # CONFIG_NOA1305 is not set
 # CONFIG_OPT3001 is not set
 # CONFIG_PA12203001 is not set
@@ -5196,8 +5126,8 @@ CONFIG_MAX44009=m
 # CONFIG_TSL4531 is not set
 # CONFIG_US5182D is not set
 # CONFIG_VCNL4000 is not set
-CONFIG_VCNL4035=m
-CONFIG_VEML6030=m
+# CONFIG_VCNL4035 is not set
+# CONFIG_VEML6030 is not set
 # CONFIG_VEML6070 is not set
 # CONFIG_VL6180 is not set
 # CONFIG_ZOPT2201 is not set
@@ -5216,9 +5146,8 @@ CONFIG_HID_SENSOR_MAGNETOMETER_3D=m
 # CONFIG_IIO_ST_MAGN_3AXIS is not set
 # CONFIG_SENSORS_HMC5843_I2C is not set
 # CONFIG_SENSORS_HMC5843_SPI is not set
-CONFIG_SENSORS_RM3100=m
-CONFIG_SENSORS_RM3100_I2C=m
-CONFIG_SENSORS_RM3100_SPI=m
+# CONFIG_SENSORS_RM3100_I2C is not set
+# CONFIG_SENSORS_RM3100_SPI is not set
 # end of Magnetometer sensors
 
 #
@@ -5256,7 +5185,7 @@ CONFIG_HID_SENSOR_DEVICE_ROTATION=m
 # CONFIG_MCP4018 is not set
 # CONFIG_MCP4131 is not set
 # CONFIG_MCP4531 is not set
-CONFIG_MCP41010=m
+# CONFIG_MCP41010 is not set
 # CONFIG_TPL0102 is not set
 # end of Digital potentiometers
 
@@ -5275,7 +5204,7 @@ CONFIG_MCP41010=m
 # CONFIG_DPS310 is not set
 CONFIG_HID_SENSOR_PRESS=m
 # CONFIG_HP03 is not set
-CONFIG_ICP10100=m
+# CONFIG_ICP10100 is not set
 # CONFIG_MPL115_I2C is not set
 # CONFIG_MPL115_SPI is not set
 # CONFIG_MPL3115 is not set
@@ -5298,15 +5227,15 @@ CONFIG_ICP10100=m
 #
 # CONFIG_ISL29501 is not set
 # CONFIG_LIDAR_LITE_V2 is not set
-CONFIG_MB1232=m
+# CONFIG_MB1232 is not set
 # CONFIG_PING is not set
 # CONFIG_RFD77402 is not set
 # CONFIG_SRF04 is not set
-CONFIG_SX9310=m
+# CONFIG_SX9310 is not set
 # CONFIG_SX9500 is not set
 # CONFIG_SRF08 is not set
-CONFIG_VCNL3020=m
-CONFIG_VL53L0X_I2C=m
+# CONFIG_VCNL3020 is not set
+# CONFIG_VL53L0X_I2C is not set
 # end of Proximity and distance sensors
 
 #
@@ -5319,7 +5248,7 @@ CONFIG_VL53L0X_I2C=m
 #
 # Temperature sensors
 #
-CONFIG_LTC2983=m
+# CONFIG_LTC2983 is not set
 # CONFIG_MAXIM_THERMOCOUPLE is not set
 CONFIG_HID_SENSOR_TEMP=m
 # CONFIG_MLX90614 is not set
@@ -5328,7 +5257,7 @@ CONFIG_HID_SENSOR_TEMP=m
 # CONFIG_TMP007 is not set
 # CONFIG_TSYS01 is not set
 # CONFIG_TSYS02D is not set
-CONFIG_MAX31856=m
+# CONFIG_MAX31856 is not set
 # end of Temperature sensors
 
 CONFIG_NTB=m
@@ -5380,8 +5309,7 @@ CONFIG_PWM_LPSS_PLATFORM=m
 
 CONFIG_RAS=y
 # CONFIG_RAS_CEC is not set
-CONFIG_USB4=m
-# CONFIG_USB4_DEBUGFS_WRITE is not set
+# CONFIG_USB4 is not set
 
 #
 # Android
@@ -5398,8 +5326,6 @@ CONFIG_BTT=y
 CONFIG_NVDIMM_KEYS=y
 CONFIG_DAX_DRIVER=y
 CONFIG_DAX=y
-CONFIG_DEV_DAX_HMEM=m
-CONFIG_DEV_DAX_HMEM_DEVICES=y
 CONFIG_NVMEM=y
 CONFIG_NVMEM_SYSFS=y
 
@@ -5412,20 +5338,10 @@ CONFIG_NVMEM_SYSFS=y
 
 # CONFIG_FPGA is not set
 # CONFIG_TEE is not set
-CONFIG_MULTIPLEXER=m
-
-#
-# Multiplexer drivers
-#
-CONFIG_MUX_ADG792A=m
-# CONFIG_MUX_ADGS1408 is not set
-CONFIG_MUX_GPIO=m
-# end of Multiplexer drivers
-
 # CONFIG_UNISYS_VISORBUS is not set
 # CONFIG_SIOX is not set
 # CONFIG_SLIMBUS is not set
-CONFIG_INTERCONNECT=y
+# CONFIG_INTERCONNECT is not set
 # CONFIG_COUNTER is not set
 # CONFIG_MOST is not set
 # end of Device Drivers
@@ -5434,7 +5350,7 @@ CONFIG_INTERCONNECT=y
 # File systems
 #
 CONFIG_DCACHE_WORD_ACCESS=y
-CONFIG_VALIDATE_FS_PARSER=y
+# CONFIG_VALIDATE_FS_PARSER is not set
 CONFIG_FS_IOMAP=y
 # CONFIG_EXT2_FS is not set
 # CONFIG_EXT3_FS is not set
@@ -5462,7 +5378,6 @@ CONFIG_GFS2_FS_LOCKING_DLM=y
 # CONFIG_BTRFS_FS is not set
 # CONFIG_NILFS2_FS is not set
 # CONFIG_F2FS_FS is not set
-# CONFIG_ZONEFS_FS is not set
 CONFIG_FS_DAX=y
 CONFIG_FS_POSIX_ACL=y
 CONFIG_EXPORTFS=y
@@ -5470,9 +5385,7 @@ CONFIG_EXPORTFS_BLOCK_OPS=y
 CONFIG_FILE_LOCKING=y
 # CONFIG_MANDATORY_FILE_LOCKING is not set
 # CONFIG_FS_ENCRYPTION is not set
-CONFIG_FS_VERITY=y
-# CONFIG_FS_VERITY_DEBUG is not set
-# CONFIG_FS_VERITY_BUILTIN_SIGNATURES is not set
+# CONFIG_FS_VERITY is not set
 CONFIG_FSNOTIFY=y
 CONFIG_DNOTIFY=y
 CONFIG_INOTIFY_USER=y
@@ -5489,8 +5402,7 @@ CONFIG_AUTOFS4_FS=y
 CONFIG_AUTOFS_FS=y
 CONFIG_FUSE_FS=m
 CONFIG_CUSE=m
-CONFIG_VIRTIO_FS=m
-CONFIG_FUSE_DAX=y
+# CONFIG_VIRTIO_FS is not set
 CONFIG_OVERLAY_FS=m
 # CONFIG_OVERLAY_FS_REDIRECT_DIR is not set
 # CONFIG_OVERLAY_FS_REDIRECT_ALWAYS_FOLLOW is not set
@@ -5531,8 +5443,7 @@ CONFIG_FAT_DEFAULT_CODEPAGE=437
 CONFIG_FAT_DEFAULT_IOCHARSET="ascii"
 # CONFIG_VFAT_NO_CREATE_WITH_LONGNAMES is not set
 # CONFIG_FAT_DEFAULT_UTF8 is not set
-CONFIG_EXFAT_FS=m
-CONFIG_EXFAT_DEFAULT_IOCHARSET="utf8"
+# CONFIG_EXFAT_FS is not set
 # CONFIG_NTFS_FS is not set
 # end of DOS/FAT/EXFAT/NT Filesystems
 
@@ -5547,13 +5458,12 @@ CONFIG_PROC_SYSCTL=y
 CONFIG_PROC_PAGE_MONITOR=y
 CONFIG_PROC_CHILDREN=y
 CONFIG_PROC_PID_ARCH_STATUS=y
-CONFIG_PROC_CPU_RESCTRL=y
 CONFIG_KERNFS=y
 CONFIG_SYSFS=y
 CONFIG_TMPFS=y
 CONFIG_TMPFS_POSIX_ACL=y
 CONFIG_TMPFS_XATTR=y
-CONFIG_TMPFS_INODE64=y
+# CONFIG_TMPFS_INODE64 is not set
 CONFIG_HUGETLBFS=y
 CONFIG_HUGETLB_PAGE=y
 CONFIG_MEMFD_CREATE=y
@@ -5614,13 +5524,7 @@ CONFIG_PSTORE_COMPRESS_DEFAULT="deflate"
 CONFIG_PSTORE_RAM=m
 # CONFIG_SYSV_FS is not set
 # CONFIG_UFS_FS is not set
-CONFIG_EROFS_FS=m
-# CONFIG_EROFS_FS_DEBUG is not set
-CONFIG_EROFS_FS_XATTR=y
-CONFIG_EROFS_FS_POSIX_ACL=y
-CONFIG_EROFS_FS_SECURITY=y
-CONFIG_EROFS_FS_ZIP=y
-CONFIG_EROFS_FS_CLUSTER_PAGE_LIMIT=1
+# CONFIG_EROFS_FS is not set
 # CONFIG_AUFS_FS is not set
 CONFIG_NETWORK_FILESYSTEMS=y
 CONFIG_NFS_FS=m
@@ -5652,7 +5556,7 @@ CONFIG_NFSD_PNFS=y
 # CONFIG_NFSD_BLOCKLAYOUT is not set
 CONFIG_NFSD_SCSILAYOUT=y
 # CONFIG_NFSD_FLEXFILELAYOUT is not set
-CONFIG_NFSD_V4_2_INTER_SSC=y
+# CONFIG_NFSD_V4_2_INTER_SSC is not set
 # CONFIG_NFSD_V4_SECURITY_LABEL is not set
 CONFIG_GRACE_PERIOD=m
 CONFIG_LOCKD=m
@@ -5668,7 +5572,7 @@ CONFIG_RPCSEC_GSS_KRB5=m
 CONFIG_CEPH_FS=m
 # CONFIG_CEPH_FSCACHE is not set
 CONFIG_CEPH_FS_POSIX_ACL=y
-CONFIG_CEPH_FS_SECURITY_LABEL=y
+# CONFIG_CEPH_FS_SECURITY_LABEL is not set
 CONFIG_CIFS=m
 # CONFIG_CIFS_STATS2 is not set
 CONFIG_CIFS_ALLOW_INSECURE_LEGACY=y
@@ -5736,8 +5640,7 @@ CONFIG_NLS_KOI8_U=m
 CONFIG_NLS_UTF8=m
 CONFIG_DLM=m
 CONFIG_DLM_DEBUG=y
-CONFIG_UNICODE=y
-# CONFIG_UNICODE_NORMALIZATION_SELFTEST is not set
+# CONFIG_UNICODE is not set
 CONFIG_IO_WQ=y
 # end of File systems
 
@@ -5745,12 +5648,11 @@ CONFIG_IO_WQ=y
 # Security options
 #
 CONFIG_KEYS=y
-CONFIG_KEYS_REQUEST_CACHE=y
+# CONFIG_KEYS_REQUEST_CACHE is not set
 # CONFIG_PERSISTENT_KEYRINGS is not set
 CONFIG_TRUSTED_KEYS=m
 CONFIG_ENCRYPTED_KEYS=y
 # CONFIG_KEY_DH_OPERATIONS is not set
-CONFIG_KEY_NOTIFICATIONS=y
 # CONFIG_SECURITY_DMESG_RESTRICT is not set
 CONFIG_SECURITY=y
 CONFIG_SECURITY_WRITABLE_HOOKS=y
@@ -5781,16 +5683,12 @@ CONFIG_SECURITY_SELINUX_SID2STR_CACHE_SIZE=256
 # CONFIG_SECURITY_LOADPIN is not set
 CONFIG_SECURITY_YAMA=y
 # CONFIG_SECURITY_SAFESETID is not set
-CONFIG_SECURITY_LOCKDOWN_LSM=y
-CONFIG_SECURITY_LOCKDOWN_LSM_EARLY=y
-CONFIG_LOCK_DOWN_KERNEL_FORCE_NONE=y
-# CONFIG_LOCK_DOWN_KERNEL_FORCE_INTEGRITY is not set
-# CONFIG_LOCK_DOWN_KERNEL_FORCE_CONFIDENTIALITY is not set
+# CONFIG_SECURITY_LOCKDOWN_LSM is not set
 # CONFIG_INTEGRITY is not set
 # CONFIG_IMA_SECURE_AND_OR_TRUSTED_BOOT is not set
 CONFIG_DEFAULT_SECURITY_SELINUX=y
 # CONFIG_DEFAULT_SECURITY_DAC is not set
-CONFIG_LSM="yama,loadpin,safesetid,integrity,selinux,smack,tomoyo,apparmor"
+CONFIG_LSM="lockdown,yama,loadpin,safesetid,integrity,selinux,smack,tomoyo,apparmor,bpf"
 
 #
 # Kernel hardening options
@@ -5832,7 +5730,7 @@ CONFIG_CRYPTO_RNG_DEFAULT=y
 CONFIG_CRYPTO_AKCIPHER2=y
 CONFIG_CRYPTO_AKCIPHER=y
 CONFIG_CRYPTO_KPP2=y
-CONFIG_CRYPTO_KPP=y
+CONFIG_CRYPTO_KPP=m
 CONFIG_CRYPTO_ACOMP2=y
 CONFIG_CRYPTO_MANAGER=y
 CONFIG_CRYPTO_MANAGER2=y
@@ -5853,13 +5751,13 @@ CONFIG_CRYPTO_GLUE_HELPER_X86=y
 # Public-key cryptography
 #
 CONFIG_CRYPTO_RSA=y
-CONFIG_CRYPTO_DH=y
+CONFIG_CRYPTO_DH=m
 CONFIG_CRYPTO_ECC=m
 CONFIG_CRYPTO_ECDH=m
-CONFIG_CRYPTO_ECRDSA=m
+# CONFIG_CRYPTO_ECRDSA is not set
 # CONFIG_CRYPTO_SM2 is not set
-CONFIG_CRYPTO_CURVE25519=m
-CONFIG_CRYPTO_CURVE25519_X86=m
+# CONFIG_CRYPTO_CURVE25519 is not set
+# CONFIG_CRYPTO_CURVE25519_X86 is not set
 
 #
 # Authenticated Encryption with Associated Data
@@ -5876,19 +5774,18 @@ CONFIG_CRYPTO_ECHAINIV=m
 # Block modes
 #
 CONFIG_CRYPTO_CBC=y
-CONFIG_CRYPTO_CFB=m
+CONFIG_CRYPTO_CFB=y
 CONFIG_CRYPTO_CTR=y
-CONFIG_CRYPTO_CTS=y
+CONFIG_CRYPTO_CTS=m
 CONFIG_CRYPTO_ECB=y
-CONFIG_CRYPTO_LRW=y
-CONFIG_CRYPTO_OFB=m
+CONFIG_CRYPTO_LRW=m
+# CONFIG_CRYPTO_OFB is not set
 CONFIG_CRYPTO_PCBC=m
-CONFIG_CRYPTO_XTS=y
+CONFIG_CRYPTO_XTS=m
 # CONFIG_CRYPTO_KEYWRAP is not set
-CONFIG_CRYPTO_NHPOLY1305=m
-CONFIG_CRYPTO_NHPOLY1305_SSE2=m
-CONFIG_CRYPTO_NHPOLY1305_AVX2=m
-CONFIG_CRYPTO_ADIANTUM=m
+# CONFIG_CRYPTO_NHPOLY1305_SSE2 is not set
+# CONFIG_CRYPTO_NHPOLY1305_AVX2 is not set
+# CONFIG_CRYPTO_ADIANTUM is not set
 CONFIG_CRYPTO_ESSIV=m
 
 #
@@ -5906,10 +5803,10 @@ CONFIG_CRYPTO_CRC32C=y
 CONFIG_CRYPTO_CRC32C_INTEL=m
 CONFIG_CRYPTO_CRC32=m
 CONFIG_CRYPTO_CRC32_PCLMUL=m
-CONFIG_CRYPTO_XXHASH=y
-CONFIG_CRYPTO_BLAKE2B=y
-CONFIG_CRYPTO_BLAKE2S=m
-CONFIG_CRYPTO_BLAKE2S_X86=m
+# CONFIG_CRYPTO_XXHASH is not set
+# CONFIG_CRYPTO_BLAKE2B is not set
+# CONFIG_CRYPTO_BLAKE2S is not set
+# CONFIG_CRYPTO_BLAKE2S_X86 is not set
 CONFIG_CRYPTO_CRCT10DIF=y
 CONFIG_CRYPTO_CRCT10DIF_PCLMUL=m
 CONFIG_CRYPTO_GHASH=y
@@ -5927,10 +5824,10 @@ CONFIG_CRYPTO_SHA1_SSSE3=y
 CONFIG_CRYPTO_SHA256_SSSE3=y
 CONFIG_CRYPTO_SHA512_SSSE3=m
 CONFIG_CRYPTO_SHA256=y
-CONFIG_CRYPTO_SHA512=y
+CONFIG_CRYPTO_SHA512=m
 CONFIG_CRYPTO_SHA3=m
-CONFIG_CRYPTO_SM3=m
-CONFIG_CRYPTO_STREEBOG=m
+# CONFIG_CRYPTO_SM3 is not set
+# CONFIG_CRYPTO_STREEBOG is not set
 CONFIG_CRYPTO_TGR192=m
 CONFIG_CRYPTO_WP512=m
 CONFIG_CRYPTO_GHASH_CLMUL_NI_INTEL=m
@@ -5997,7 +5894,7 @@ CONFIG_CRYPTO_USER_API_RNG=y
 # CONFIG_CRYPTO_USER_API_RNG_CAVP is not set
 CONFIG_CRYPTO_USER_API_AEAD=y
 # CONFIG_CRYPTO_USER_API_ENABLE_OBSOLETE is not set
-CONFIG_CRYPTO_STATS=y
+# CONFIG_CRYPTO_STATS is not set
 CONFIG_CRYPTO_HASH_INFO=y
 
 #
@@ -6005,27 +5902,22 @@ CONFIG_CRYPTO_HASH_INFO=y
 #
 CONFIG_CRYPTO_LIB_AES=y
 CONFIG_CRYPTO_LIB_ARC4=m
-CONFIG_CRYPTO_ARCH_HAVE_LIB_BLAKE2S=m
-CONFIG_CRYPTO_LIB_BLAKE2S_GENERIC=m
-CONFIG_CRYPTO_LIB_BLAKE2S=m
+# CONFIG_CRYPTO_LIB_BLAKE2S is not set
 CONFIG_CRYPTO_ARCH_HAVE_LIB_CHACHA=m
 CONFIG_CRYPTO_LIB_CHACHA_GENERIC=m
-CONFIG_CRYPTO_LIB_CHACHA=m
-CONFIG_CRYPTO_ARCH_HAVE_LIB_CURVE25519=m
-CONFIG_CRYPTO_LIB_CURVE25519_GENERIC=m
-CONFIG_CRYPTO_LIB_CURVE25519=m
+# CONFIG_CRYPTO_LIB_CHACHA is not set
+# CONFIG_CRYPTO_LIB_CURVE25519 is not set
 CONFIG_CRYPTO_LIB_DES=m
 CONFIG_CRYPTO_LIB_POLY1305_RSIZE=11
 CONFIG_CRYPTO_ARCH_HAVE_LIB_POLY1305=m
 CONFIG_CRYPTO_LIB_POLY1305_GENERIC=m
-CONFIG_CRYPTO_LIB_POLY1305=m
-CONFIG_CRYPTO_LIB_CHACHA20POLY1305=m
+# CONFIG_CRYPTO_LIB_POLY1305 is not set
+# CONFIG_CRYPTO_LIB_CHACHA20POLY1305 is not set
 CONFIG_CRYPTO_LIB_SHA256=y
 CONFIG_CRYPTO_HW=y
 # CONFIG_CRYPTO_DEV_PADLOCK is not set
-CONFIG_CRYPTO_DEV_ATMEL_I2C=m
-CONFIG_CRYPTO_DEV_ATMEL_ECC=m
-CONFIG_CRYPTO_DEV_ATMEL_SHA204A=m
+# CONFIG_CRYPTO_DEV_ATMEL_ECC is not set
+# CONFIG_CRYPTO_DEV_ATMEL_SHA204A is not set
 CONFIG_CRYPTO_DEV_CCP=y
 CONFIG_CRYPTO_DEV_CCP_DD=m
 CONFIG_CRYPTO_DEV_SP_CCP=y
@@ -6045,10 +5937,9 @@ CONFIG_CRYPTO_DEV_NITROX_CNN55XX=m
 # CONFIG_CRYPTO_DEV_AMLOGIC_GXL is not set
 CONFIG_ASYMMETRIC_KEY_TYPE=y
 CONFIG_ASYMMETRIC_PUBLIC_KEY_SUBTYPE=y
-CONFIG_ASYMMETRIC_TPM_KEY_SUBTYPE=m
+# CONFIG_ASYMMETRIC_TPM_KEY_SUBTYPE is not set
 CONFIG_X509_CERTIFICATE_PARSER=y
-CONFIG_PKCS8_PRIVATE_KEY_PARSER=m
-CONFIG_TPM_KEY_PARSER=m
+# CONFIG_PKCS8_PRIVATE_KEY_PARSER is not set
 CONFIG_PKCS7_MESSAGE_PARSER=y
 # CONFIG_PKCS7_TEST_KEY is not set
 CONFIG_SIGNED_PE_FILE_VERIFICATION=y
@@ -6060,8 +5951,7 @@ CONFIG_BINARY_PRINTF=y
 #
 CONFIG_RAID6_PQ=m
 CONFIG_RAID6_PQ_BENCHMARK=y
-CONFIG_LINEAR_RANGES=m
-CONFIG_PACKING=y
+# CONFIG_PACKING is not set
 CONFIG_BITREVERSE=y
 CONFIG_GENERIC_STRNCPY_FROM_USER=y
 CONFIG_GENERIC_STRNLEN_USER=y
@@ -6085,10 +5975,10 @@ CONFIG_CRC32_SLICEBY8=y
 # CONFIG_CRC32_SLICEBY4 is not set
 # CONFIG_CRC32_SARWATE is not set
 # CONFIG_CRC32_BIT is not set
-CONFIG_CRC64=m
+# CONFIG_CRC64 is not set
 # CONFIG_CRC4 is not set
 CONFIG_CRC7=m
-CONFIG_LIBCRC32C=y
+CONFIG_LIBCRC32C=m
 CONFIG_CRC8=m
 CONFIG_XXHASH=y
 # CONFIG_RANDOM32_SELFTEST is not set
@@ -6177,7 +6067,7 @@ CONFIG_SBITMAP=y
 CONFIG_PRINTK_TIME=y
 # CONFIG_PRINTK_CALLER is not set
 CONFIG_CONSOLE_LOGLEVEL_DEFAULT=7
-CONFIG_CONSOLE_LOGLEVEL_QUIET=3
+CONFIG_CONSOLE_LOGLEVEL_QUIET=4
 CONFIG_MESSAGE_LOGLEVEL_DEFAULT=4
 CONFIG_BOOT_PRINTK_DELAY=y
 CONFIG_DYNAMIC_DEBUG=y
@@ -6200,7 +6090,7 @@ CONFIG_ENABLE_MUST_CHECK=y
 CONFIG_FRAME_WARN=2048
 CONFIG_STRIP_ASM_SYMS=y
 # CONFIG_READABLE_ASM is not set
-CONFIG_HEADERS_INSTALL=y
+# CONFIG_HEADERS_INSTALL is not set
 # CONFIG_DEBUG_SECTION_MISMATCH is not set
 CONFIG_SECTION_MISMATCH_WARN_ONLY=y
 CONFIG_STACK_VALIDATION=y
@@ -6277,8 +6167,8 @@ CONFIG_BOOTPARAM_SOFTLOCKUP_PANIC_VALUE=0
 CONFIG_HARDLOCKUP_DETECTOR_PERF=y
 CONFIG_HARDLOCKUP_CHECK_TIMESTAMP=y
 CONFIG_HARDLOCKUP_DETECTOR=y
-# CONFIG_BOOTPARAM_HARDLOCKUP_PANIC is not set
-CONFIG_BOOTPARAM_HARDLOCKUP_PANIC_VALUE=0
+CONFIG_BOOTPARAM_HARDLOCKUP_PANIC=y
+CONFIG_BOOTPARAM_HARDLOCKUP_PANIC_VALUE=1
 CONFIG_DETECT_HUNG_TASK=y
 CONFIG_DEFAULT_HUNG_TASK_TIMEOUT=120
 # CONFIG_BOOTPARAM_HUNG_TASK_PANIC is not set
@@ -6368,7 +6258,7 @@ CONFIG_TRACING=y
 CONFIG_GENERIC_TRACER=y
 CONFIG_TRACING_SUPPORT=y
 CONFIG_FTRACE=y
-CONFIG_BOOTTIME_TRACING=y
+# CONFIG_BOOTTIME_TRACING is not set
 CONFIG_FUNCTION_TRACER=y
 CONFIG_FUNCTION_GRAPH_TRACER=y
 CONFIG_DYNAMIC_FTRACE=y
-- 
2.17.1

